// @aircall/tractor/lib/components/Modal/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
var _exportNames = {
  Modal: true
};
exports.Modal = void 0;

var _Modal = require("./Modal");

var _ModalBody = require("./ModalBody");

var _ModalClose = require("./ModalClose");

var _ModalFooter = require("./ModalFooter");

var _ModalHeader = require("./ModalHeader");

var _ModalTitle = require("./ModalTitle");

var _Modal2 = require("./Modal.decl");

Object.keys(_Modal2).forEach(function (key) {
  if (key === "default" || key === "__esModule") return;
  if (Object.prototype.hasOwnProperty.call(_exportNames, key)) return;
  if (key in exports && exports[key] === _Modal2[key]) return;
  Object.defineProperty(exports, key, {
    enumerable: true,
    get: function get() {
      return _Modal2[key];
    }
  });
});
var Modal = {
  Dialog: _Modal.ModalDialog,
  Header: _ModalHeader.ModalHeader,
  Body: _ModalBody.ModalBody,
  Footer: _ModalFooter.ModalFooter,
  Title: _ModalTitle.ModalTitle,
  Close: _ModalClose.ModalClose
};
exports.Modal = Modal;
