// @aircall/tractor/lib/components/Modal/Modal.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.ModalDialog = void 0;

var _react = _interopRequireDefault(require("react"));

var _framerMotion = require("framer-motion");

var _reactDom = require("react-dom");

var _context = require("./context");

var _hooks = require("./hooks");

var _styled = require("./styled");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var ModalDialog = function ModalDialog(_ref) {
  var show = _ref.show,
      onHide = _ref.onHide,
      _ref$rootClose = _ref.rootClose,
      rootClose = _ref$rootClose === void 0 ? true : _ref$rootClose,
      children = _ref.children,
      _ref$size = _ref.size,
      size = _ref$size === void 0 ? 'regular' : _ref$size,
      _ref$portalContainer = _ref.portalContainer,
      portalContainer = _ref$portalContainer === void 0 ? document.body : _ref$portalContainer;
  (0, _hooks.useCloseModalOnEscape)(show, onHide);
  (0, _hooks.useUpdateBodyClassName)(show);
  return /*#__PURE__*/_react["default"].createElement(_context.ModalContext.Provider, {
    value: {
      onHide: onHide
    }
  }, portalContainer && /*#__PURE__*/(0, _reactDom.createPortal)( /*#__PURE__*/_react["default"].createElement(_framerMotion.AnimatePresence, {
    "data-test": "animatePresence"
  }, show && /*#__PURE__*/_react["default"].createElement(_styled.ModalContainer, {
    "data-test": "modal-container"
  }, /*#__PURE__*/_react["default"].createElement(_styled.ModalBackdrop, {
    "data-test": "modal-backdrop",
    onClick: !rootClose ? undefined : onHide
  }), /*#__PURE__*/_react["default"].createElement(_styled.ModalDialogWrapper, null, /*#__PURE__*/_react["default"].createElement(_styled.ModalDialogContent, {
    size: size,
    "data-test": "modal-dialog-content",
    mx: [2, 4, 4],
    overflow: "hidden",
    backgroundColor: "grey.lighter",
    borderRadius: 8,
    flexDirection: "column"
  }, children)))), portalContainer));
};

exports.ModalDialog = ModalDialog;
