// @aircall/tractor/lib/components/Modal/ModalHeader.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.ModalHeader = void 0;

var _react = _interopRequireDefault(require("react"));

var _Box = require("../Box");

var _Flex = require("../Flex");

var _ModalClose = require("./ModalClose");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var ModalHeader = function ModalHeader(_ref) {
  var _ref$closeButton = _ref.closeButton,
      closeButton = _ref$closeButton === void 0 ? true : _ref$closeButton,
      children = _ref.children;
  return /*#__PURE__*/_react["default"].createElement(_Flex.Flex, {
    "data-test": "modal-header",
    position: "sticky",
    top: "0",
    left: "0",
    alignItems: "center",
    justifyContent: "space-between",
    pl: "m",
    borderBottomWidth: "1px",
    borderBottomStyle: "solid",
    borderBottomColor: "grey.light",
    backgroundColor: "base.white"
  }, /*#__PURE__*/_react["default"].createElement(_Box.Box, null, children), closeButton && /*#__PURE__*/_react["default"].createElement(_ModalClose.ModalClose, null));
};

exports.ModalHeader = ModalHeader;
