// @aircall/tractor/lib/components/Modal/styled.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.dialogAnimation = exports.containerAnimation = exports.ModalDialogWrapper = exports.ModalDialogContent = exports.ModalContainer = exports.ModalBackdrop = void 0;

var _polished = require("polished");

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _ = require("..");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var containerAnimation = {
  enter: {
    opacity: 1,
    transition: {
      when: 'beforeChildren',
      duration: 0.25
    }
  },
  exit: {
    opacity: 0,
    transition: {
      delay: 0.25,
      duration: 0.25
    }
  }
};
exports.containerAnimation = containerAnimation;
var ModalContainer = (0, _styledComponents["default"])(_.MotionBox).attrs({
  variants: containerAnimation,
  initial: 'exit',
  animate: 'enter',
  exit: 'exit',
  py: 'xl'
}).withConfig({
  displayName: "styled__ModalContainer",
  componentId: "sc-148di7t-0"
})(["position:fixed;z-index:1000;top:0;right:0;bottom:0;left:0;overflow:auto;"]);
exports.ModalContainer = ModalContainer;
var dialogAnimation = {
  enter: {
    y: 0,
    opacity: 1,
    transition: {
      duration: 0.25
    }
  },
  exit: {
    y: 100,
    opacity: 0,
    transition: {
      duration: 0.25
    }
  }
};
exports.dialogAnimation = dialogAnimation;
var ModalDialogWrapper = (0, _styledComponents["default"])(_.MotionBox).attrs({
  variants: dialogAnimation
}).withConfig({
  displayName: "styled__ModalDialogWrapper",
  componentId: "sc-148di7t-1"
})(["display:flex;max-height:100%;justify-content:center;"]);
exports.ModalDialogWrapper = ModalDialogWrapper;

var ModalBackdrop = _styledComponents["default"].div.withConfig({
  displayName: "styled__ModalBackdrop",
  componentId: "sc-148di7t-2"
})(["position:absolute;z-index:0;top:0;right:0;bottom:0;left:0;background:", ";"], function (_ref) {
  var theme = _ref.theme;
  return (0, _polished.transparentize)(0.35, theme.colors.secondary.base);
});

exports.ModalBackdrop = ModalBackdrop;
var ModalDialogContent = (0, _styledComponents["default"])(_.Flex).attrs(function (_ref2) {
  var theme = _ref2.theme,
      size = _ref2.size;
  return {
    width: theme.modal.sizes[size].width
  };
}).withConfig({
  displayName: "styled__ModalDialogContent",
  componentId: "sc-148di7t-3"
})(["z-index:1;"]);
exports.ModalDialogContent = ModalDialogContent;
