// @aircall/tractor/lib/components/Flag/components/Atf.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgAtf(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M16.667.01v16.657H0a39.247 39.247 0 01.003-.684c-.033-2.492.2-5.316.863-8.108 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c.222.002.445.005.667.01z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M5.333 1.705V16H.003v-.017c-.033-2.492.2-5.316.863-8.108 0 0 .87-4.158 4.467-6.17z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M16 0v16h-5.334V.314A50.525 50.525 0 0116 0z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M19.6 17.219l.889 1.415h2.528v4.935L21.119 20.6l-2.554 4.072h1.242l.251-.449h2.118l1.634 2.934 1.635-2.934h2.118l.25.449h1.243L26.502 20.6l-1.898 2.968v-2.727h1.139l.846-1.345h-1.985v-.863h2.528l.889-1.415H19.6zm7.423 6.039l-.52-.932-.52.932h1.04zm-5.385 0h-1.04l.52-.932.52.932z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M18.2 20.813l-.901.628.318-1.051-.875-.664 1.098-.022.36-1.037.36 1.037 1.098.022-.875.664.318 1.05-.901-.627zm11.334 0l-.902.628.318-1.051-.875-.664 1.098-.022.36-1.037.361 1.037 1.098.022-.875.664.318 1.05-.901-.627zm-8.867 7l-.902.628.319-1.051-.875-.664 1.097-.022.36-1.037.361 1.037 1.098.022-.875.664.318 1.05-.901-.627zm6.266 0l-.9.628.317-1.051-.875-.664 1.098-.022.36-1.037.36 1.037 1.099.022-.875.664.318 1.05-.902-.627zm-3.067 2.334l-.9.627.317-1.051-.875-.663 1.098-.023.36-1.037.361 1.037 1.098.023-.875.663.318 1.05-.902-.626z",
    fill: "#fff"
  }));
}

var _default = SvgAtf;
exports["default"] = _default;
