// @aircall/tractor/lib/components/Flag/components/Brn.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgBrn(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M3.253 3.367l28.434 17.77c-.135.99-.317 1.99-.554 2.987 0 0-.514 2.458-2.387 4.51L.313 10.861c.134-.989.316-1.99.553-2.987 0 0 .514-2.457 2.387-4.508z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M1.212 6.758l29.576 18.484c-.34.904-.967 2.215-2.042 3.391L.313 10.863c.134-.99.316-1.99.553-2.988 0 0 .094-.446.346-1.117z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M18.256 21.41c.153-.082-.284.422.3.59.585.168 1.13-.967 2.051-.447.184-.148-.051-.437-.374-.458.132-.278.112-.4.08-.448-.316-.45-1.08.1-1.012.043a.467.467 0 10-.602-.714c-.87.734-1.766 1.092-2.698 1.09-.936.002-1.831-.356-2.702-1.09a.467.467 0 00-.601.714c.066.056-.697-.493-1.012-.043-.033.048-.053.17.08.448-.323.021-.559.31-.375.458.922-.52 1.466.615 2.051.447.585-.168.147-.672.3-.59a4.659 4.659 0 002.257.59 4.658 4.658 0 002.257-.59z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M20.667 16.186a.5.5 0 01.122-.327l.528-.612a.504.504 0 00.07-.106l.242-.492a.5.5 0 01.351-.27l.402-.08a.2.2 0 01.236.234l-.184.979a.5.5 0 01-.105.225l-.548.666a.5.5 0 00-.114.317v1.517c0 .08.02.16.056.231l.192.367a.181.181 0 01-.079.251 1.66 1.66 0 01-.67.172 1.66 1.66 0 01-.668-.172.181.181 0 01-.079-.25l.191-.368a.5.5 0 00.057-.23v-2.052zm-9.327 0a.5.5 0 00-.122-.327l-.529-.612a.5.5 0 01-.07-.106l-.242-.492a.5.5 0 00-.35-.27l-.403-.08a.2.2 0 00-.236.234l.184.979a.5.5 0 00.106.225l.547.666a.5.5 0 01.114.317v1.517a.5.5 0 01-.056.231l-.191.367c-.049.093-.017.207.078.251.16.075.415.172.67.172.254 0 .51-.097.669-.172a.18.18 0 00.078-.25l-.191-.368a.5.5 0 01-.057-.23v-2.052zM16 12.667a.667.667 0 100-1.333.667.667 0 000 1.333z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M19.507 13.325c-.438.834-1.752 1.576-2.84 1.343l-.34.717a.333.333 0 00-.024.215l.214.969c.014.06.01.122-.01.18l-.108.316a.333.333 0 00.02.262l.562 1.08c-.21.431-.536.647-.98.648-.445 0-.772-.217-.981-.648l.563-1.08a.333.333 0 00.02-.262l-.11-.316a.333.333 0 01-.01-.18l.215-.969a.333.333 0 00-.024-.215l-.34-.717c-1.088.233-2.402-.509-2.84-1.343 1.347.37 2.544-1.682 3.506-.175.963-1.507 2.16.545 3.507.175z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M12.483 14.76a3.667 3.667 0 107.034 0 4 4 0 11-7.034 0z",
    fill: "#E94B35"
  }));
}

var _default = SvgBrn;
exports["default"] = _default;
