// @aircall/tractor/lib/components/Flag/components/Cpv.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgCpv(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.835 24h30.327l-.029.124s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124L.836 24z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.062 18.667h31.87a34.056 34.056 0 01-.272 2.666H.325a31.92 31.92 0 01-.263-2.666z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.997 16H.003v-.017c-.033-2.492.2-5.316.863-8.108 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.662 2.752.862 5.57.83 8.091z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10.02 15.015l.98-.682.98.682-.346-1.142.951-.721-1.193-.025L11 12l-.392 1.127-1.193.025.951.72-.346 1.143zm3.118.376l1.193.025.392 1.127.392-1.127 1.193-.025-.951-.72.345-1.143-.98.682-.979-.682.346 1.142-.951.721zm3.251 2.858l-.95-.721 1.193-.024.391-1.128.392 1.127 1.194.025-.952.72.346 1.143-.98-.681-.98.681.346-1.142zm-.345 2.36l.345 1.142-.95.721 1.193.024.391 1.128.392-1.127 1.194-.025-.952-.72.346-1.143-.98.681-.98-.681zm-1.713 3.975l.392-1.127.392 1.127 1.193.025-.951.72.345 1.143-.98-.682-.979.682.346-1.142-.951-.721 1.193-.025zm-2.351.401l-.98.682-.98-.682.346 1.142-.951.721 1.193.025L11 28l.392-1.127 1.193-.025-.95-.72.345-1.143zm-4.31-.4l1.193.024-.952.72.346 1.143-.98-.682-.98.682.346-1.142-.95-.721 1.193-.025.391-1.127.392 1.127zm-1.108-2.113l-.951-.72.345-1.143-.98.681-.979-.681.346 1.142-.951.721 1.193.024.392 1.128.392-1.127 1.193-.025zM5.61 18.25l.345 1.142-.98-.681-.979.681.346-1.142-.951-.721 1.193-.024.392-1.128.392 1.127 1.193.025-.951.72zm1.666-1.706l.392-1.127 1.194-.025-.952-.72.346-1.143-.98.682-.98-.682.346 1.142-.95.721 1.193.025.391 1.127z",
    fill: "#F2C500"
  }));
}

var _default = SvgCpv;
exports["default"] = _default;
