// @aircall/tractor/lib/components/Flag/components/Dma.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgDma(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M15 .018V13h16.92c.044.679.069 1.347.077 2H15v16.982a54.307 54.307 0 01-2-.089V15H.004c.01-.65.038-1.32.087-2h12.91V.106a54.17 54.17 0 012-.088z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M17 .018V15h14.997c.004.344.004.683 0 1.016.004.323.004.651-.001.984H17v14.983A62.88 62.88 0 0116 32a57.464 57.464 0 01-1-.018V17H.003a39.42 39.42 0 010-1.017C0 15.661 0 15.333.004 15H15V.018c.333-.01.667-.015 1-.018.333.003.667.009 1 .018z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M19 .107V17h12.995c-.01.65-.037 1.32-.086 2h-12.91v12.899c-.656.04-1.326.067-2 .084V19H.083a36.284 36.284 0 01-.08-2H17V.018c.673.017 1.343.047 2 .09z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M19.684 22.349a7.385 7.385 0 01-7.368 0A7.263 7.263 0 018.666 16a7.263 7.263 0 013.65-6.349 7.385 7.385 0 017.368 0A7.263 7.263 0 0123.333 16a7.263 7.263 0 01-3.65 6.349z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M15.412 11.142l.588-.409.588.41-.207-.686.57-.433-.716-.014L16 9.333l-.235.677-.716.014.57.433-.207.685zm2.968.583l.716.014.235.677.235-.677.716-.014-.57-.433.207-.685-.588.409-.588-.41.207.686-.57.433zm2.629 2.648l-.57-.433.715-.015.235-.676.236.676.715.015-.57.433.207.685-.588-.41-.587.41.207-.685zm-.207 2.57l.207.684-.57.433.715.015.235.676.236-.676.715-.015-.57-.432.207-.686-.588.41-.587-.41zm-1.706 3.318l.235-.677.235.677.716.015-.57.432.207.686-.588-.41-.588.41.207-.686-.57-.432.716-.015zm-2.508.597l-.588.409-.588-.41.208.686-.571.433.716.014.235.677.235-.677.716-.014-.57-.433.207-.685zm-3.684-.597l.716.015-.57.432.207.686-.588-.41-.588.41.208-.686-.57-.432.715-.015.235-.677.235.677zm-1.342-2.2l-.57-.433.206-.686-.587.41-.588-.41.207.686-.57.432.716.015.235.676.235-.676.716-.015zm-.57-3.688l.206.685-.587-.41-.588.41.207-.685-.57-.433.716-.015.235-.676.235.676.716.015-.57.433zm1.677-1.957l.235-.677.716-.014-.57-.433.207-.685-.588.409-.588-.41.208.686-.57.433.715.014.235.677z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M15.266 13.184s-.34-.55-.354-.57c-.115-.153.138-.377.354-.5.118-.065.554-.16.961.046.437.22.83.81 1.03.83-.081.088-.366-.015-.478-.033.2.13.429.333.6.274-.122.138-.074.206-.235.2.197.076.314.204.463.177-.072.13-.269.23-.269.23s.184.185.37.13c-.07.126-.247.222-.247.222s.039.184.344.15c-.37.546-1.158 4.034-1.472 3.915-.314-.118-1.455-1.059-1.65-2.255-.198-1.226.583-2.816.583-2.816z",
    fill: "#9C56B8"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M15.159 12.862c.044.02.21.126.034.29-.11.102-.395.133-.46.08-.075.062-.067.14-.099.243-.29-.452-.104-.825.275-.893.325-.082.25.28.25.28z",
    fill: "#E5B731"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M18.088 19.053c.044-.17.065-1.478-.188-2.59-.215-.943-.682-1.7-.924-1.972-.149-.167-.442-.354-.607-.515-.09-.188-.073-.717-.225-.717-.135 0-.521.053-.899-.035a1.344 1.344 0 00-.222.587c.07.321.633.323.829.438-.128.93.517 2.604.517 2.604s-.305.23-.225 1.322c-.17-.124-.77-.814-.673-.594-.104.965 1.558 2.62 1.845 2.785.529.06.584-1.711.584-1.711s.172.33.188.398z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M14.767 18.706c.055.139.01.276.081.277.073.002 2.982-.075 3.052.042.07.116.08.346.007.436-.074.091-3.763.126-3.848.032-.085-.093-.209-.486-.208-.51.26-.018.233-.235.365-.348.132-.113.495-.068.55.07z",
    fill: "#9C4A00"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16.33 19.717c.004.022-.037.175-.247 0-.02.167-.498-.018-.534-.389-.022-.392.438-.694.462-.471 0 .006.242-.012.446 0 .176.01.32.049.322.074.019.175-.22.113-.202.341.018.229.337.209.356.399.01.149-.28.088-.392-.033.08.124-.11.233-.211.08z",
    fill: "#FCD116"
  }));
}

var _default = SvgDma;
exports["default"] = _default;
