// @aircall/tractor/lib/components/Flag/components/Gib.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgGib(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.325 21.333H6v-.666h1.333V12H6v-1.333h2V6h-.667V4.667H12V6h-.667v4.667h2V4h-.667V2.667h6.667V4h-.666v6.667h2V6H20V4.667h4.666V6H24v4.667h2V12h-1.334v8.667H26v.666h5.66a30.81 30.81 0 01-.527 2.791s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.232-.93-.41-1.865-.54-2.79z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M9.666 7.333a1 1 0 011 1v2.334h-2V8.333a1 1 0 011-1zM16 6.667c.736 0 1.333.597 1.333 1.333v2.667h-2.666V8c0-.736.597-1.333 1.333-1.333zm6.334.666a1 1 0 011 1v2.334h-2V8.333a1 1 0 011-1zM10 16c.736 0 1.333.597 1.333 1.333v2H8.666v-2C8.666 16.597 9.263 16 10 16zm6-1.333a2 2 0 012 2v2.666h-4v-2.666a2 2 0 012-2zM22 16c.736 0 1.333.597 1.333 1.333v2h-2.666v-2c0-.736.596-1.333 1.333-1.333z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16.442 20.237c.356-.433 1.36-.866 1.437-.893v-.57c-.15.051-1.491.856-1.72 1.126-.447-.333-1.697-.944-2.092-1.145l-.018.501c.337.18 1.372.601 1.736.98-.223.103-.364.289-.364.535v1.096l-1.153.73h-.321v.559h.321l1.481.937v3.485h-1.802v1.056h.572v-.53h.779v.53h.451v.351h-.451v.529h-.779v-.529h-.572v1.056h1.802v.704h.492V23.99l1.316-.833h.322v-.56h-.322l-.825-.522v-1.303c0-.375-.21-.424-.29-.534z",
    fill: "#F2C500"
  }));
}

var _default = SvgGib;
exports["default"] = _default;
