// @aircall/tractor/lib/components/Flag/components/Jey.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgJey(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M27.711 29.598L2.78 3.928a8.836 8.836 0 011.509-1.526l24.931 25.67a8.831 8.831 0 01-1.509 1.526z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M2.78 28.071L27.707 2.406a8.804 8.804 0 011.517 1.518L4.29 29.598a8.835 8.835 0 01-1.51-1.527zM16 12.333c-3.019-1.736-3.385-4.864-2.717-7.397.712-.606 3.946-.98 5.434 0 .668 2.533.302 5.661-2.717 7.397z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M18.9 5.333c.126-.9.573-1.8 1.1-2.481-.55-.01-.797.256-.802.534-.138-.356-.677-.335-.843-.142.602.372.138 1.032-.407.896a.554.554 0 01-.438-.464c.15-.008.27-.12.277-.257.007-.138-.101-.258-.25-.28-.15-.02-.292.064-.331.197-.04.133.037.272.178.322a.669.669 0 01-.702.277.523.523 0 01-.407-.518c0-.544.573-.628.745-.534-.011-.366-.642-.66-.836-.356A.917.917 0 0016 1.333a.917.917 0 00-.183 1.194c-.195-.304-.825-.01-.837.356.172-.094.745-.01.745.534 0 .21-.12.445-.407.518a.67.67 0 01-.701-.277c.14-.05.217-.189.178-.322a.296.296 0 00-.331-.196c-.15.02-.257.141-.25.279.006.138.126.249.276.257a.554.554 0 01-.438.464c-.545.136-1.009-.524-.407-.896-.166-.193-.705-.214-.843.142-.006-.278-.252-.545-.802-.534.527.68.974 1.58 1.1 2.481.642-.366 1.73-.649 2.9-.649s2.258.283 2.9.65z",
    fill: "#F2C500"
  }));
}

var _default = SvgJey;
exports["default"] = _default;
