// @aircall/tractor/lib/components/Flag/components/Kor.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgKor(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M4.313 10.002l2.61-3.868.553.372-2.61 3.87-.553-.374zm1.106.746l2.61-3.869.552.373-2.61 3.869-.552-.373zm1.105.746l2.61-3.87.553.374-2.61 3.868-.553-.372zm.553 8.64l2.61 3.868-.553.373-2.61-3.869.553-.372zm-1.105.745l1.118 1.658-.553.373-1.118-1.658.553-.373zm1.49 2.211l1.12 1.658-.554.373-1.118-1.658.553-.373zm-2.596-1.465l2.61 3.869-.553.372-2.61-3.868.553-.373zM25.077 6.134l1.118 1.658-.552.372-1.119-1.658.553-.372zm1.491 2.21l1.118 1.658-.552.373-1.119-1.658.553-.373zM23.972 6.88l2.61 3.869-.554.373-2.61-3.869.554-.373zm-1.106.746l1.119 1.658-.553.373-1.118-1.658.552-.373zm1.491 2.21l1.119 1.659-.553.372-1.118-1.658.552-.372zm-2.043 14.166l1.118-1.658.553.373-1.119 1.658-.552-.373zm1.49-2.21l1.119-1.658.553.372-1.119 1.659-.552-.373zm-.385 2.956l1.118-1.658.553.373-1.118 1.658-.553-.373zm1.491-2.21l1.118-1.659.553.373-1.118 1.658-.553-.373zm-.386 2.956l1.119-1.658.552.372-1.118 1.658-.553-.372zm1.491-2.211l1.119-1.658.552.373-1.118 1.658-.553-.373z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 21.333a5.333 5.333 0 100-10.666 5.333 5.333 0 000 10.666z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M21.333 16a5.333 5.333 0 01-10.666 0A2.667 2.667 0 0016 16a2.667 2.667 0 115.333 0z",
    fill: "#227FBB"
  }));
}

var _default = SvgKor;
exports["default"] = _default;
