// @aircall/tractor/lib/components/Flag/components/Lka.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgLka(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.866 1.333c2.746 1.142 4.147 3.508 4.8 5.069V25.55c-.632 1.544-2.013 3.943-4.755 5.117H10.666V1.333h15.2z",
    fill: "#9C3900"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M6.088 1.333h3.246v29.334H6.089a7.966 7.966 0 01-.756-.372V1.705c.24-.134.491-.259.755-.372z",
    fill: "#E87E04"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M5.333 1.705v28.59c-2.245-1.257-3.428-3.349-4-4.744V6.45c.572-1.396 1.755-3.488 4-4.744z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M13 4.667h1v-1a1 1 0 10-1 1zm15.067-1.964c.467.406.868.845 1.212 1.291a1 1 0 01-.945.673h-1v-1a1 1 0 01.733-.964zM13 27.333h1v1a1 1 0 11-1-1zm16.276.666c-.341.448-.74.89-1.205 1.3a1 1 0 01-.738-.966v-1h1a1 1 0 01.943.666zm-11.467-8.823a.667.667 0 00-.667-.667h-.36a.615.615 0 01-.196-.03c-.45-.152-2.329-.83-2.329-1.608 0-.611.294-1.354.475-1.754a.75.75 0 00.017-.588c-.267-.663-.86-2.267-.721-3.096.084-.506.49-.932.892-1.239.384-.292.877.014.877.496v3.782c0 .168.063.329.176.452l.788.855c.268.292.73.286.992-.012l.177-.203c.272-.31.2-.79-.151-1.007l-1.098-.677v-.517a.5.5 0 00.2-.927l-.2-.118v-.885h1.493c.177 0 .346-.07.471-.195l.494-.494a.667.667 0 01.472-.195h1.053c.267 0 .508.159.613.404l.576 1.347a.667.667 0 01-.023.572l-.534 1.016a.667.667 0 00.567.976l2.793.1a.667.667 0 00.412-.125l.205-.146a.667.667 0 00-.028-1.103l-.439-.282a.667.667 0 00-.36-.106h-.907a.667.667 0 01-.666-.666v-.695c0-.25.14-.48.362-.594l.99-.507-.282.644c-.193.44.13.934.611.934h.03a.62.62 0 01.114.01l1.29.222c.157.027.299.11.4.232l.777.941a.645.645 0 01.148.36c.042.518.133 1.905-.018 2.657-.137.688-.549 1.318-.759 1.606a.752.752 0 00-.145.37l-.268 2.794a.594.594 0 11-1.139-.285l.32-.769a.421.421 0 00-.011-.35l-.043-.086c-.188-.376-.755-.242-.755.179 0 .052-.01.104-.031.153l-.537 1.288a.663.663 0 11-1.204-.552l.335-.669a.667.667 0 00.07-.298v-.945a.667.667 0 00-.667-.667h-2.46a.667.667 0 00-.666.667v.945a.667.667 0 01-.07.298l-.63 1.26a.667.667 0 01-.597.368h-.035a.667.667 0 01-.596-.964l.332-.664a.666.666 0 00.07-.298v-.945z",
    fill: "#F2C500"
  }));
}

var _default = SvgLka;
exports["default"] = _default;
