// @aircall/tractor/lib/components/Flag/components/Mys.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgMys(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M2.258 27.333h27.484a9.16 9.16 0 01-2.034 2.267H4.292a9.161 9.161 0 01-2.034-2.267zm-1.69-4.555H31.42c-.085.449-.181.898-.288 1.346 0 0-.075.36-.273.92H1.14c-.198-.56-.274-.92-.274-.92a26.337 26.337 0 01-.298-1.346zm-.529-4.556h31.917c-.037.74-.098 1.499-.187 2.267H.217a33.593 33.593 0 01-.178-2.267zm31.918-4.555c.038.798.05 1.579.04 2.333H16v-2.333h15.957zm-.53-4.556c.146.757.26 1.515.346 2.267H16V9.11h15.427zm-1.746-4.555c.555.834.922 1.649 1.152 2.266H16V4.556h13.681zm-2.159-2.289H16V0c3.75.033 7.567.432 9.69 1.263.69.268 1.298.612 1.832 1.004z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 0v16H.003v-.017c-.033-2.492.2-5.316.863-8.108 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M9.498 4.29a4 4 0 100 7.42 4.667 4.667 0 110-7.42z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10.617 6.382L11 5l.383 1.382.967-1.078-.271 1.375 1.314-.567-.861 1.16L14 7.325 12.704 8 14 8.675l-1.468.053.86 1.16-1.313-.567.27 1.375-.966-1.078L11 11l-.383-1.382-.967 1.078.271-1.375-1.314.567.861-1.16L8 8.675 9.296 8 8 7.325l1.468-.053-.86-1.16 1.313.567-.27-1.375.966 1.078z",
    fill: "#F2C500"
  }));
}

var _default = SvgMys;
exports["default"] = _default;
