// @aircall/tractor/lib/components/Flag/components/Ncl.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgNcl(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.325 21.333H31.66a30.81 30.81 0 01-.527 2.791s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.232-.93-.41-1.865-.54-2.79z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.34 10.667h31.343c.254 1.844.336 3.663.314 5.35a34.717 34.717 0 01-.337 5.316H.325a34.916 34.916 0 01-.322-5.35c-.022-1.66.074-3.469.337-5.316z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.683 10.667H.34c.131-.925.305-1.86.526-2.792 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.22.916.39 1.84.516 2.758z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10 24a8 8 0 100-16 8 8 0 000 16z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M7.252 23.516A7.984 7.984 0 0010 24a7.98 7.98 0 002.749-.485c-.676-1.012-1.593-1.517-2.75-1.515-1.155.002-2.07.507-2.747 1.516zM10 21.667c.368 0 .667-.597.667-1.334 0-.736-.299-1.333-.667-1.333-.368 0-.666.597-.666 1.333 0 .737.298 1.334.666 1.334zm0-4.334c.736 0 1.333-.298 1.333-.666 0-.369-.597-.667-1.333-.667-.737 0-1.334.299-1.334.667 0 .368.597.666 1.334.666zm0-3a1 1 0 100-2 1 1 0 000 2zm-1.334 1h2.667v.334H8.666v-.334zm0 2.334h2.667V18H8.666v-.333z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M9.17 18.333h1.66c.298 0 .56.198.642.484l.528 1.85s-1.157-2-2-2c-.843-.001-2 2-2 2l.529-1.85a.667.667 0 01.64-.484zm0-3.333h1.66c.298 0 .56-.197.642-.483l.528-1.85s-1.157 1.999-2 2c-.843 0-2-2-2-2l.529 1.85a.667.667 0 00.64.483z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10.2 8.002v14.665h-.4V8.002a8.158 8.158 0 01.4 0z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M9.333 11.86c.443-.085 1.976-2.14 1.976-2.351 0-.212-1.616 1.024-1.976 1.158-.36.134-.667.298-.667.666 0 .368.224.613.667.528z",
    fill: "#000"
  }));
}

var _default = SvgNcl;
exports["default"] = _default;
