// @aircall/tractor/lib/components/Flag/components/Slv.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgSlv(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.34 10.667h31.343c.254 1.844.336 3.663.314 5.35a34.717 34.717 0 01-.337 5.316H.325a34.914 34.914 0 01-.322-5.35c-.022-1.66.074-3.469.337-5.316z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.683 10.667H.34c.131-.925.305-1.86.526-2.792 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.22.916.39 1.84.516 2.758zM.325 21.333H31.66a30.81 30.81 0 01-.527 2.791s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.232-.93-.41-1.865-.54-2.79z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M15.776 11.94a4.138 4.138 0 01.63.013.267.267 0 00.052-.53 4.692 4.692 0 00-.711-.016.267.267 0 10.029.532zm-.836.133a4.033 4.033 0 00-.589.208.267.267 0 01-.217-.487c.216-.096.44-.175.668-.236a.267.267 0 11.138.515zm-1.85 1.087a4.1 4.1 0 01.47-.413.267.267 0 10-.321-.427 4.633 4.633 0 00-.532.468.267.267 0 00.382.372zm-.545.694c-.11.178-.207.363-.289.556a.267.267 0 01-.49-.21c.092-.217.201-.427.326-.627a.267.267 0 01.453.281zm-.61 2.024c.006-.21.028-.418.066-.622a.267.267 0 00-.524-.097 4.608 4.608 0 00-.075.704.267.267 0 10.533.015zm.066.866c.038.206.092.408.161.605a.267.267 0 01-.503.177 4.575 4.575 0 01-.182-.685.267.267 0 11.524-.097zm.941 1.937a4.086 4.086 0 01-.375-.5.267.267 0 10-.45.286c.126.2.268.388.424.566a.267.267 0 00.401-.352zm1.181.928a4.079 4.079 0 01-.53-.331.267.267 0 10-.317.43c.19.14.391.265.6.374a.267.267 0 00.247-.473zm3.72-7.235a4.037 4.037 0 00-.577-.24.267.267 0 01.166-.507c.224.074.443.165.653.272a.267.267 0 01-.242.475zm.772.511c.16.135.31.281.447.439a.267.267 0 00.401-.352 4.632 4.632 0 00-.505-.495.267.267 0 10-.343.408zm1.205 1.717a4.053 4.053 0 00-.26-.569.266.266 0 11.467-.258c.114.207.213.422.294.644a.267.267 0 01-.5.183zm.22.926c.018.156.027.313.027.472v.064a.267.267 0 00.532.008v-.036L20.6 16c0-.18-.01-.357-.03-.533a.267.267 0 00-.53.061zm-.27 2.002c.078-.193.141-.392.189-.596a.267.267 0 11.52.122 4.64 4.64 0 01-.216.674.267.267 0 01-.494-.2zm-.436.799a4.08 4.08 0 01-.397.484.267.267 0 10.385.369c.164-.171.314-.354.45-.547a.267.267 0 10-.438-.306zm-1.444 1.273c.13-.069.256-.144.378-.226a.267.267 0 11.298.443c-.138.092-.28.178-.428.255a.267.267 0 11-.248-.472z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M13.777 13.515a3.333 3.333 0 104.445 0 .333.333 0 10-.445.497 2.667 2.667 0 11-3.555 0 .333.333 0 10-.445-.497z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M13.334 14.667L16 16l2.667-1.333v.666s-.2 3.334-2.667 2c-2.466 1.334-2.666-2-2.666-2v-.666z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M14 13.333l2 2.024 2-2.024V14s-.15 3.333-2 2c-1.85 1.333-2-2-2-2v-.667z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 13.333l2.333 4h-4.666l2.333-4z",
    fill: "#D9D9D9"
  }));
}

var _default = SvgSlv;
exports["default"] = _default;
