// @aircall/tractor/lib/components/Flag/components/Svn.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgSvn(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.683 10.667H.34c.131-.925.305-1.86.526-2.792 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.22.916.39 1.84.516 2.758z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.34 10.667h31.343c.254 1.844.336 3.663.314 5.35a34.717 34.717 0 01-.337 5.316H.325a34.914 34.914 0 01-.322-5.35c-.022-1.66.074-3.469.337-5.316z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.325 21.333H31.66a30.81 30.81 0 01-.527 2.791s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.232-.93-.41-1.865-.54-2.79z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M12.162 11.676l.42-6.35a11.824 11.824 0 00-10.35 0l.42 6.35c.19 2.854 2.083 5.29 4.756 6.116 2.672-.827 4.566-3.263 4.754-6.116z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M3.026 11.65c.175 2.653 1.911 4.927 4.382 5.741 2.47-.814 4.206-3.088 4.382-5.74l.43-6.494C12.274 5.184 10.05 4 7.407 4 4.79 4 2.53 5.189 2.596 5.157l.43 6.493z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M3.526 12.537c.45 2.074 1.92 3.76 3.881 4.451 1.962-.691 3.432-2.377 3.882-4.451l-1.941-2.65-.776 1.06L7.407 8.56l-1.164 2.385-.776-1.06-1.94 2.65",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M3.7 13.2a1.3 1.3 0 00.786.617c.331.09.684.043.981-.133.4-.236.894-.236 1.294 0 .4.237.894.237 1.294 0 .4-.236.893-.236 1.294 0 .297.176.65.224.981.133.332-.091.614-.313.786-.618v.383a1.299 1.299 0 01-.786.617 1.264 1.264 0 01-.981-.132 1.267 1.267 0 00-1.294 0c-.4.237-.894.237-1.294 0a1.267 1.267 0 00-1.294 0c-.297.176-.65.223-.981.132a1.298 1.298 0 01-.786-.617v-.383z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M3.7 13.964c.172.305.454.527.786.618.331.09.684.043.981-.133.4-.236.894-.236 1.294 0 .4.237.894.237 1.294 0 .4-.236.893-.236 1.294 0 .297.176.65.224.981.133.332-.091.614-.313.786-.618v.383a1.299 1.299 0 01-.786.617 1.267 1.267 0 01-.981-.132 1.267 1.267 0 00-1.294 0c-.4.237-.894.237-1.294 0a1.267 1.267 0 00-1.294 0c-.297.176-.65.223-.981.132a1.298 1.298 0 01-.786-.617v-.383z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M7.408 6.739l.097.324.323-.076-.226.249.226.248-.323-.076-.097.325-.097-.325-.323.076.226-.248-.226-.249.323.076.097-.324zM5.952 4.75l.097.326.323-.077-.226.249.226.248-.323-.076-.097.325-.097-.325-.323.076.226-.248-.226-.249.323.077.097-.325zm2.911 0l.097.326.323-.077-.226.249.226.248-.323-.076-.097.325-.097-.325-.323.076.226-.248-.226-.249.323.077.097-.325z",
    fill: "#F2C500"
  }));
}

var _default = SvgSvn;
exports["default"] = _default;
