// @aircall/tractor/lib/components/Flag/components/Tkm.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var React = _interopRequireWildcard(require("react"));

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function SvgTkm(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10 .399V31.6c-1.456-.2-2.727-.476-3.658-.83C4.176 29.927 2.82 28.328 2 26.91V5.088c.82-1.418 2.176-3.018 4.342-3.86C7.272.875 8.544.6 10 .4z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M6 6.667c1.84 0 3.333-1.045 3.333-2.334C9.333 3.045 7.841 2 6 2 4.159 2 2.667 3.045 2.667 4.333c0 1.289 1.492 2.334 3.333 2.334z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M2.692 4.042A8.812 8.812 0 014.48 2.256 4.51 4.51 0 016 2v2.333h3.333c0 1.289-1.492 2.334-3.333 2.334V4.333H2.667c0-.098.008-.195.025-.29z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M6 5.333c.736 0 1.333-.447 1.333-1 0-.552-.597-1-1.333-1-.737 0-1.333.448-1.333 1 0 .553.596 1 1.333 1z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M4.48 29.744a8.812 8.812 0 01-1.788-1.787 1.66 1.66 0 01-.026-.29c0-1.289 1.493-2.334 3.334-2.334 1.84 0 3.333 1.045 3.333 2.334C9.333 28.955 7.841 30 6 30a4.51 4.51 0 01-1.52-.256z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M2.667 27.667c0-1.289 1.492-2.334 3.333-2.334v2.334h3.333C9.333 28.955 7.841 30 6 30v-2.333H2.667z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M6 12.667c1.84 0 3.333-1.045 3.333-2.334C9.333 9.045 7.841 8 6 8c-1.841 0-3.333 1.045-3.333 2.333 0 1.289 1.492 2.334 3.333 2.334z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M5.333 8.047v4.573A4.118 4.118 0 014 12.2V8.467c.388-.204.84-.35 1.333-.42zM8 8.467V12.2c-.388.204-.84.35-1.333.42V8.047c.492.07.945.216 1.333.42z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M6 11.333c1.105 0 2-.447 2-1 0-.552-.895-1-2-1s-2 .448-2 1c0 .553.895 1 2 1z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M6 18c-1.841 0-3.333-.895-3.333-2S4.159 14 6 14c1.84 0 3.333.895 3.333 2S7.841 18 6 18zm0-1.333c1.104 0 2-.299 2-.667 0-.368-.896-.667-2-.667-1.105 0-2 .299-2 .667 0 .368.895.667 2 .667zM6 24c1.84 0 3.333-1.045 3.333-2.333 0-1.289-1.492-2.334-3.333-2.334-1.841 0-3.333 1.045-3.333 2.334C2.667 22.955 4.159 24 6 24z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M6 21.333A.667.667 0 106 20a.667.667 0 000 1.333zm-2 1A.667.667 0 104 21a.667.667 0 000 1.333zm2 1A.667.667 0 106 22a.667.667 0 000 1.333zm2-1A.667.667 0 108 21a.667.667 0 000 1.333z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M6 28.667c.736 0 1.333-.448 1.333-1 0-.553-.597-1-1.333-1-.737 0-1.333.447-1.333 1 0 .552.596 1 1.333 1z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M13.621 10.379a5.333 5.333 0 007.424-7.424 5.333 5.333 0 11-7.424 7.424zm2.836-6.999l-.433.571-.015-.716L15.333 3l.676-.235.015-.716.433.57.685-.207-.409.588.41.588-.686-.208zm0 2.667l-.433.57-.015-.715-.676-.235.676-.235.015-.716.433.57.685-.207-.409.588.41.587-.686-.207z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16.457 8.714l-.433.57-.015-.716-.676-.235.676-.235.015-.716.433.57.685-.206-.409.587.41.588-.686-.207zm-2.667-2l-.432.57-.015-.716-.676-.235.676-.235.015-.716.432.57.686-.206-.41.587.41.588-.686-.207zm5.334-1.334l-.433.571-.014-.716L18 5l.677-.235.014-.716.433.57.685-.207L19.4 5l.41.588-.686-.208z",
    fill: "#fff"
  }));
}

var _default = SvgTkm;
exports["default"] = _default;
