// @aircall/tractor/lib/components/Tag/Tag.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var _react = _interopRequireWildcard(require("react"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _styledSystem = require("styled-system");

var _styles = require("../../utils/styles");

var _Icon = require("../Icon");

var _Spacer = require("../Spacer");

var _Tag = require("./Tag.decl");

var _utils = require("./utils");

var _excluded = ["children", "size", "mode", "variant", "maxWidth", "deletable", "onDelete", "onClick"];

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

var TagIcon = (0, _styledComponents["default"])(_Icon.Icon).withConfig({
  displayName: "Tag__TagIcon",
  componentId: "sc-1wyvv20-0"
})(["", ""], _utils.makeTagIconShape);
var TagContainer = (0, _styledComponents["default"])(_Spacer.Spacer).withConfig({
  shouldForwardProp: _utils.shouldForwardProp
}).withConfig({
  displayName: "Tag__TagContainer",
  componentId: "sc-1wyvv20-1"
})(["", " ", " ", " ", " ", " ", ""], _utils.makeTagShape, _styles.makeBackgroundVariant, _styles.makeTextColorVariant, function (_ref) {
  var tagSize = _ref.tagSize,
      theme = _ref.theme;
  return tagSize === _Tag.TagSize.NORMAL ? theme.typography.variants.body : theme.typography.variants.caption;
}, function (_ref2) {
  var onClick = _ref2.onClick;
  return onClick && 'cursor: pointer;';
}, _styledSystem.color);
var Tag = /*#__PURE__*/(0, _react.forwardRef)(function (_ref3, ref) {
  var children = _ref3.children,
      _ref3$size = _ref3.size,
      size = _ref3$size === void 0 ? _Tag.TagSize.NORMAL : _ref3$size,
      _ref3$mode = _ref3.mode,
      mode = _ref3$mode === void 0 ? 'light' : _ref3$mode,
      _ref3$variant = _ref3.variant,
      variant = _ref3$variant === void 0 ? 'primary' : _ref3$variant,
      _ref3$maxWidth = _ref3.maxWidth,
      maxWidth = _ref3$maxWidth === void 0 ? '260px' : _ref3$maxWidth,
      _ref3$deletable = _ref3.deletable,
      deletable = _ref3$deletable === void 0 ? false : _ref3$deletable,
      onDelete = _ref3.onDelete,
      onClick = _ref3.onClick,
      remainingStyleProps = _objectWithoutPropertiesLoose(_ref3, _excluded);

  var tagSpace = size === _Tag.TagSize.NORMAL ? 's' : 'xxs';
  return /*#__PURE__*/_react["default"].createElement(TagContainer, _extends({
    ref: ref,
    variant: variant,
    mode: mode,
    tagSize: size,
    alignItems: "center",
    space: tagSpace,
    onClick: onClick
  }, remainingStyleProps), children, !!deletable && /*#__PURE__*/_react["default"].createElement(TagIcon, {
    component: _Icon.CloseOutlined,
    "area-role": "button",
    tagSize: size,
    size: "16px",
    mode: mode,
    variant: variant,
    onClick: onDelete
  }));
});
var _default = Tag;
exports["default"] = _default;
