// @aircall/tractor/lib/components/SquareButton/utils.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.square = exports.shouldForwardProp = exports.makeSquareButtonVariant = void 0;

var _styledSystem = require("styled-system");

var _SquareButton = require("./SquareButton.decl");

var makeSquareButtonVariant = function makeSquareButtonVariant(_ref) {
  var disabled = _ref.disabled,
      theme = _ref.theme,
      variant = _ref.variant;
  var colors = theme.colors;

  if (disabled) {
    return {
      backgroundColor: colors.grey.light,
      borderColor: colors.grey.light,
      color: colors.grey.dark,
      svg: {
        fill: colors.grey.dark
      }
    };
  }

  if (variant !== _SquareButton.SquareButtonVariant.DEFAULT) {
    return;
  }

  return {
    backgroundColor: colors.base.white,
    borderColor: colors.base.white,
    svg: {
      fill: colors.secondary.base
    },
    ':hover': {
      borderColor: colors.grey.lighter,
      svg: {
        fill: colors.secondary.base
      }
    },
    ':active': {
      borderColor: colors.grey.base,
      svg: {
        fill: colors.secondary.base
      }
    },
    ':focus': {
      borderColor: colors.grey.lighter,
      svg: {
        fill: colors.secondary.base
      }
    }
  };
};

exports.makeSquareButtonVariant = makeSquareButtonVariant;
var square = (0, _styledSystem.system)({
  square: {
    properties: ['width', 'height'],
    scale: 'sizes'
  }
}); // Props that should not be forwarded to the HTML DOM elements

exports.square = square;

var shouldForwardProp = function shouldForwardProp(prop) {
  return !['square'].includes(prop);
};

exports.shouldForwardProp = shouldForwardProp;
