// @aircall/tractor/lib/components/Typography/Typography.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _styledSystem = require("styled-system");

var _utils = require("../../utils");

var _Box = require("../Box");

var _Typography = require("./Typography.decl");

var _utils2 = require("./utils");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var customProps = (0, _styledSystem.system)({
  wordBreak: true,
  whiteSpace: true
});
var Typography = (0, _styledComponents["default"])(_Box.Box).withConfig({
  shouldForwardProp: _utils2.shouldForwardProp
}).withConfig({
  displayName: "Typography",
  componentId: "sc-1ucmpew-0"
})(["font-family:", ";overflow-wrap:break-word;", ";", " ", " ", " ", ""], (0, _utils.fromTheme)('typography.fontFamily'), _utils2.makeTypographyVariant, _utils2.makeEllipsis, _styledSystem.typography, _styledSystem.color, customProps);
Typography.defaultProps = {
  variant: _Typography.TypographyVariant.BODY
};
var _default = Typography;
exports["default"] = _default;
