// @aircall/tractor/lib/components/Avatar/Avatar.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var _react = _interopRequireWildcard(require("react"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _styledSystem = require("styled-system");

var _styles = require("../../utils/styles");

var _Flex = require("../Flex");

var _Avatar = require("./Avatar.decl");

var _utils = require("./utils");

var _excluded = ["size", "src", "icon", "disabled", "children", "mode", "variant", "onClick", "borderless", "color"];

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

var AvatarContainer = (0, _styledComponents["default"])(_Flex.Flex).withConfig({
  shouldForwardProp: _utils.shouldForwardProp
}).withConfig({
  displayName: "Avatar__AvatarContainer",
  componentId: "sc-15mkrx3-0"
})(["position:relative;align-items:center;justify-content:center;flex-shrink:0;user-select:none;font-family:", ";clip-path:url(#svgPath);", " ", " ", " ", " &::after{content:'';position:absolute;background:", ";clip-path:url(#svgPath);top:1px;left:1px;right:1px;bottom:1px;z-index:-1;", " ", "}", ""], function (_ref) {
  var theme = _ref.theme;
  return theme.typography.fontFamily;
}, _utils.makeAvatarSize, _utils.makeAvatarIconSize, _utils.makeAvatarColor, _styles.makeTextColorVariant, function (_ref2) {
  var theme = _ref2.theme;
  return theme.colors.base.white;
}, _styles.makeBackgroundVariant, _styledSystem.color, _styledSystem.color);

var AvatarImg = _styledComponents["default"].img.withConfig({
  displayName: "Avatar__AvatarImg",
  componentId: "sc-15mkrx3-1"
})(["position:absolute;width:100%;height:100%;object-fit:cover;"]);

var Avatar = /*#__PURE__*/(0, _react.forwardRef)(function (_ref3, ref) {
  var _ref3$size = _ref3.size,
      size = _ref3$size === void 0 ? _Avatar.AvatarSize.MEDIUM : _ref3$size,
      src = _ref3.src,
      icon = _ref3.icon,
      _ref3$disabled = _ref3.disabled,
      disabled = _ref3$disabled === void 0 ? false : _ref3$disabled,
      children = _ref3.children,
      _ref3$mode = _ref3.mode,
      mode = _ref3$mode === void 0 ? 'light' : _ref3$mode,
      variant = _ref3.variant,
      onClick = _ref3.onClick,
      _ref3$borderless = _ref3.borderless,
      borderless = _ref3$borderless === void 0 ? false : _ref3$borderless,
      color = _ref3.color,
      otherStyleProps = _objectWithoutPropertiesLoose(_ref3, _excluded);

  var _useState = (0, _react.useState)(false),
      imgLoadError = _useState[0],
      setImgLoadError = _useState[1];

  var clickable = !!onClick;
  (0, _react.useEffect)(function () {
    setImgLoadError(false);
  }, [src]);
  var isValidSrc = src && !imgLoadError;
  return /*#__PURE__*/_react["default"].createElement(AvatarContainer, _extends({
    ref: ref,
    "data-test": "avatar",
    avatarSize: size,
    disabled: disabled,
    mode: mode,
    variant: variant,
    color: color,
    hasImage: !!isValidSrc,
    clickable: clickable,
    borderless: borderless,
    onClick: onClick
  }, otherStyleProps), isValidSrc ? /*#__PURE__*/_react["default"].createElement(AvatarImg, {
    src: src,
    onError: function onError() {
      return setImgLoadError(true);
    },
    disabled: disabled
  }) : icon || children);
});
var _default = Avatar;
exports["default"] = _default;
