// @aircall/tractor/lib/components/SquircleButton/utils.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.shouldForwardProp = exports.makeSquircleButtonVariant = exports.SQUIRCLE_BUTTON_SIZES = exports.ICON_SIZES = void 0;

var _Button = require("../Button");

var _SquircleButton = require("./SquircleButton.decl");

var _ICON_SIZES, _SQUIRCLE_BUTTON_SIZE;

var makeSquircleButtonVariant = function makeSquircleButtonVariant(_ref) {
  var theme = _ref.theme,
      variant = _ref.variant,
      disabled = _ref.disabled,
      readOnly = _ref.readOnly;
  var colors = theme.colors;

  if (variant !== _SquircleButton.SquircleButtonVariant.DEFAULT) {
    return;
  }

  if (disabled) {
    return {
      backgroundColor: colors.base.white,
      borderColor: colors.base.white,
      svg: {
        fill: colors.secondary.lighter
      }
    };
  }

  return {
    backgroundColor: colors.base.white,
    borderColor: colors.base.white,
    svg: {
      fill: colors.secondary.base
    },
    ':hover': readOnly ? undefined : {
      backgroundColor: colors.grey.lighter,
      borderColor: colors.grey.lighter,
      svg: {
        fill: colors.secondary.base
      }
    },
    ':active': readOnly ? undefined : {
      backgroundColor: colors.grey.base,
      borderColor: colors.grey.base,
      svg: {
        fill: colors.secondary.base
      }
    },
    ':focus': readOnly ? undefined : {
      backgroundColor: colors.grey.base,
      borderColor: colors.grey.lighter,
      svg: {
        fill: colors.secondary.base
      }
    }
  };
};

exports.makeSquircleButtonVariant = makeSquircleButtonVariant;
var ICON_SIZES = (_ICON_SIZES = {}, _ICON_SIZES[_Button.ButtonSize.SMALL] = '24px', _ICON_SIZES[_Button.ButtonSize.REGULAR] = '32px', _ICON_SIZES[_Button.ButtonSize.LARGE] = '40px', _ICON_SIZES);
exports.ICON_SIZES = ICON_SIZES;
var SQUIRCLE_BUTTON_SIZES = (_SQUIRCLE_BUTTON_SIZE = {}, _SQUIRCLE_BUTTON_SIZE[_Button.ButtonSize.SMALL] = {
  width: '40px',
  height: '40px'
}, _SQUIRCLE_BUTTON_SIZE[_Button.ButtonSize.REGULAR] = {
  width: '56px',
  height: '56px'
}, _SQUIRCLE_BUTTON_SIZE[_Button.ButtonSize.LARGE] = {
  width: '64px',
  height: '64px'
}, _SQUIRCLE_BUTTON_SIZE); // Props that should not be forwarded to the HTML DOM elements

exports.SQUIRCLE_BUTTON_SIZES = SQUIRCLE_BUTTON_SIZES;

var shouldForwardProp = function shouldForwardProp(prop) {
  return !['shape'].includes(prop);
};

exports.shouldForwardProp = shouldForwardProp;
