// @aircall/tractor/lib/components/TextField/TextField.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;

var _react = _interopRequireDefault(require("react"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _Icon = require("../Icon");

var _IconButton = require("../IconButton");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var LeftIcon = (0, _styledComponents["default"])(_Icon.Icon).withConfig({
  displayName: "TextField__LeftIcon",
  componentId: "sc-1ythfz5-0"
})(["position:absolute;top:50%;left:16px;transform:translateY(-50%);pointer-events:none;"]);
var ClearButton = (0, _styledComponents["default"])(_IconButton.IconButton).withConfig({
  displayName: "TextField__ClearButton",
  componentId: "sc-1ythfz5-1"
})(["position:absolute;top:50%;right:", ";background-color:transparent;border:none;transform:translateY(-50%);", ""], function (_ref) {
  var theme = _ref.theme;
  return theme.space.s + "px";
}, function (props) {
  return props.clearIconStyle;
});

var TextFieldContainer = _styledComponents["default"].div.withConfig({
  displayName: "TextField__TextFieldContainer",
  componentId: "sc-1ythfz5-2"
})(["position:relative;width:100%;"]);

function TextField(_ref2) {
  var onClear = _ref2.onClear,
      icon = _ref2.icon,
      children = _ref2.children,
      clearIconStyle = _ref2.clearIconStyle,
      error = _ref2.error;
  return /*#__PURE__*/_react["default"].createElement(TextFieldContainer, null, icon && /*#__PURE__*/_react["default"].createElement(LeftIcon, {
    component: icon
  }), _react["default"].Children.only(children), !!onClear && /*#__PURE__*/_react["default"].createElement(ClearButton, {
    type: "button",
    size: 16,
    component: _Icon.CloseOutlined,
    onClick: onClear,
    color: error ? 'red.base' : undefined,
    clearIconStyle: clearIconStyle
  }));
}

var _default = TextField;
exports["default"] = _default;
