// @aircall/tractor/lib/components/Radio/Radio.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Radio = void 0;

var _react = _interopRequireWildcard(require("react"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _common = require("../../declarations/common.decl");

var _utils = require("../../utils");

var _Flex = require("../Flex");

var _Form = require("../Form");

var _Typography = require("../Typography");

var _context = require("./context");

var _utils2 = require("./utils");

var _excluded = ["disabled", "size", "children", "value", "validationStatus"];

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

var RadioContainer = _styledComponents["default"].div.withConfig({
  shouldForwardProp: _utils2.shouldForwardProp
}).withConfig({
  displayName: "Radio__RadioContainer",
  componentId: "sc-u2rb7a-0"
})(["position:relative;display:block;box-sizing:border-box;cursor:", ";border:1px solid ", ";", " ", " transition:background 300ms,width 300ms,height 300ms;outline:0;user-select:none;border-radius:50%;input{display:none;}"], function (_ref) {
  var disabled = _ref.disabled;
  return disabled ? 'not-allowed' : 'pointer';
}, (0, _utils.fromTheme)('colors.grey.base'), _utils2.makeRadioSize, _utils2.makeRadioColor);

var RadioIndicator = _styledComponents["default"].div.withConfig({
  shouldForwardProp: _utils2.shouldForwardProp
}).withConfig({
  displayName: "Radio__RadioIndicator",
  componentId: "sc-u2rb7a-1"
})(["display:block;content:'';position:absolute;bottom:0;top:0;margin:auto;border-radius:50%;transition:transform 150ms;", " ", " ", ""], _utils2.makeRadioIndicatorColor, _utils2.makeRadioIndicatorSize, function (_ref2) {
  var checked = _ref2.checked;
  return "transform: " + (checked ? 'scale(1)' : 'scale(0)') + ";";
});

var Radio = /*#__PURE__*/(0, _react.forwardRef)(function (_ref3, ref) {
  var _ref4, _ref5;

  var disabled = _ref3.disabled,
      size = _ref3.size,
      children = _ref3.children,
      value = _ref3.value,
      validationStatus = _ref3.validationStatus,
      otherStyleProps = _objectWithoutPropertiesLoose(_ref3, _excluded);

  var radioGroupProps = (0, _context.useRadioContext)();
  var checkedValue = radioGroupProps.checkedValue;
  var checkboxCheck = checkedValue === value;
  var radioSize = (_ref4 = size != null ? size : radioGroupProps.size) != null ? _ref4 : _common.ComponentSize.REGULAR;
  var radioDisabled = (_ref5 = disabled != null ? disabled : radioGroupProps.disabled) != null ? _ref5 : false;
  var inputStatus = validationStatus != null ? validationStatus : radioGroupProps.validationStatus;
  var error = inputStatus === _Form.FormItemStatus.ERROR;
  var handleOnChange = (0, _react.useCallback)(function (event) {
    if (radioDisabled) {
      return;
    }

    radioGroupProps.onChange(value, event);
  }, [value, radioDisabled, radioGroupProps]);
  return /*#__PURE__*/_react["default"].createElement(_Flex.Flex, {
    cursor: radioDisabled ? undefined : 'pointer',
    alignItems: "center",
    as: "label"
  }, /*#__PURE__*/_react["default"].createElement(RadioContainer, _extends({
    checked: checkboxCheck,
    disabled: radioDisabled,
    size: radioSize,
    error: error
  }, otherStyleProps), /*#__PURE__*/_react["default"].createElement("input", {
    ref: ref,
    "data-test": "radio",
    type: "radio",
    value: value,
    "aria-checked": checkboxCheck,
    checked: checkboxCheck,
    disabled: radioDisabled,
    onChange: handleOnChange
  }), /*#__PURE__*/_react["default"].createElement(RadioIndicator, {
    checked: checkboxCheck,
    disabled: radioDisabled,
    size: radioSize,
    error: error
  })), /*#__PURE__*/_react["default"].createElement(_Typography.Typography, {
    ml: "xxs",
    variant: "body2",
    color: "text.base",
    userSelect: "none"
  }, children));
});
exports.Radio = Radio;
