// @aircall/tractor/lib/components/Radio/RadioGroup.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.RadioGroup = void 0;

var _react = _interopRequireWildcard(require("react"));

var _common = require("../../declarations/common.decl");

var _context = require("../Form/context");

var _Spacer = require("../Spacer");

var _context2 = require("./context");

var _excluded = ["children", "defaultValue", "value", "disabled", "validationStatus", "onChange", "size", "space"];

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

var RadioGroup = /*#__PURE__*/(0, _react.forwardRef)(function (_ref, ref) {
  var children = _ref.children,
      defaultValue = _ref.defaultValue,
      value = _ref.value,
      disabled = _ref.disabled,
      validationStatus = _ref.validationStatus,
      onChange = _ref.onChange,
      size = _ref.size,
      _ref$space = _ref.space,
      space = _ref$space === void 0 ? _Spacer.SpaceVariant.S : _ref$space,
      spacerProps = _objectWithoutPropertiesLoose(_ref, _excluded);

  var formContext = (0, _context.useFormContext)();
  var formItemContext = (0, _context.useFormItemContext)();

  var _useState = (0, _react.useState)(defaultValue),
      innerCheck = _useState[0],
      setInnerCheck = _useState[1];

  var radioSize = size || (formContext == null ? void 0 : formContext.size) || _common.ComponentSize.REGULAR;
  var inputStatus = validationStatus || (formItemContext == null ? void 0 : formItemContext.validationStatus);
  /**
   * Use the "controlled" version over the "uncontrolled" if provided
   */

  var checkedValue = value !== undefined ? value : innerCheck;
  var handleOnChange = (0, _react.useCallback)(function (value, event) {
    if (disabled) {
      return;
    }

    setInnerCheck(value);
    onChange == null ? void 0 : onChange(value, event);
  }, [disabled, onChange]);
  return /*#__PURE__*/_react["default"].createElement(_context2.RadioContext.Provider, {
    value: {
      checkedValue: checkedValue,
      onChange: handleOnChange,
      size: radioSize,
      disabled: disabled,
      validationStatus: inputStatus
    }
  }, /*#__PURE__*/_react["default"].createElement(_Spacer.Spacer, _extends({
    ref: ref,
    space: space
  }, spacerProps), children));
});
exports.RadioGroup = RadioGroup;
