// @aircall/tractor/lib/components/Accordion/Accordion.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Accordion = Accordion;

var _react = _interopRequireWildcard(require("react"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _useChildrenValidation = require("../../hooks/useChildrenValidation");

var _general = require("../../utils/general");

var _invariant = require("../../utils/invariant");

var _Flex = require("../Flex");

var _AccordionItem = require("./AccordionItem");

var _context = require("./context");

var _excluded = ["children", "selected", "defaultSelected", "onChange"];

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

var AccordionWrapper = (0, _styledComponents["default"])(_Flex.Flex).attrs({
  flexDirection: 'column',
  flexShrink: 1,
  minHeight: 0,
  maxHeight: 'auto',
  height: 'max-content'
}).withConfig({
  displayName: "Accordion__AccordionWrapper",
  componentId: "sc-zh1rup-0"
})([""]);

function Accordion(_ref) {
  var children = _ref.children,
      selected = _ref.selected,
      defaultSelected = _ref.defaultSelected,
      _ref$onChange = _ref.onChange,
      onChange = _ref$onChange === void 0 ? _general.noop : _ref$onChange,
      styleProps = _objectWithoutPropertiesLoose(_ref, _excluded);

  var controlled = typeof selected !== 'undefined';
  var uncontrolled = typeof defaultSelected !== 'undefined';
  (0, _invariant.invariant)(!(controlled && uncontrolled), "Accordion component can be either controlled or uncontrolled at a time: use either 'defaultSelected' or 'selected'.");

  var _useState = (0, _react.useState)(defaultSelected),
      internalSelectedItemId = _useState[0],
      setInternalSelectedItemId = _useState[1];

  var handleItemClick = (0, _react.useCallback)(function (clickedItem) {
    setInternalSelectedItemId(clickedItem);
    onChange == null ? void 0 : onChange(clickedItem);
  }, [setInternalSelectedItemId, onChange]);
  (0, _useChildrenValidation.useChildrenValidation)(children, 'AccordionItem', _AccordionItem.AccordionItem);
  return /*#__PURE__*/_react["default"].createElement(_context.AccordionMainContext.Provider, {
    value: {
      selectedItemId: selected != null ? selected : internalSelectedItemId,
      onItemClick: handleItemClick
    }
  }, /*#__PURE__*/_react["default"].createElement(AccordionWrapper, styleProps, children));
}
