// @aircall/tractor/lib/components/Dropdown/Dropdown.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Dropdown = void 0;

var _react = _interopRequireDefault(require("react"));

var _hooks = require("../../hooks");

var _BoundaryAlignment = require("../BoundaryAlignment");

var _constants = require("../BoundaryAlignment/constants");

var _Box = require("../Box");

var _context = require("./context");

var _utils = require("./utils");

var _excluded = ["trigger", "position", "anchor", "boundaryElement", "visible", "onVisibilityChange", "closeOnInsideClick", "children", "data-test", "body", "portalContainer", "maxWidth"];

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

var Dropdown = function Dropdown(_ref) {
  var givenTrigger = _ref.trigger,
      _ref$position = _ref.position,
      position = _ref$position === void 0 ? _constants.Position.BOTTOM : _ref$position,
      _ref$anchor = _ref.anchor,
      anchor = _ref$anchor === void 0 ? _constants.Anchor.CENTER : _ref$anchor,
      boundaryElement = _ref.boundaryElement,
      visible = _ref.visible,
      onVisibilityChange = _ref.onVisibilityChange,
      _ref$closeOnInsideCli = _ref.closeOnInsideClick,
      closeOnInsideClick = _ref$closeOnInsideCli === void 0 ? true : _ref$closeOnInsideCli,
      children = _ref.children,
      _ref$dataTest = _ref['data-test'],
      testId = _ref$dataTest === void 0 ? 'dropdown' : _ref$dataTest,
      body = _ref.body,
      portalContainer = _ref.portalContainer,
      _ref$maxWidth = _ref.maxWidth,
      maxWidth = _ref$maxWidth === void 0 ? 344 : _ref$maxWidth,
      otherProps = _objectWithoutPropertiesLoose(_ref, _excluded);

  var _useTrigger = (0, _hooks.useTrigger)(givenTrigger, {
    trigger: 'click',
    closeOnInsideClick: closeOnInsideClick,
    visible: visible,
    onVisibilityChange: onVisibilityChange,
    body: body,
    portalContainer: portalContainer
  }),
      trigger = _useTrigger[0],
      _useTrigger$ = _useTrigger[1],
      isVisible = _useTrigger$.isVisible,
      containerBoundingBox = _useTrigger$.containerBoundingBox,
      portal = _useTrigger$.portal;

  return /*#__PURE__*/_react["default"].createElement(_context.DropdownContext.Provider, {
    value: {
      isVisible: isVisible
    }
  }, trigger, isVisible && portal( /*#__PURE__*/_react["default"].createElement(_BoundaryAlignment.BoundaryAlignment, {
    containerBoundingBox: containerBoundingBox,
    boundaryElement: boundaryElement,
    position: position,
    anchor: anchor,
    offset: (0, _utils.getOffset)(position)
  }, /*#__PURE__*/_react["default"].createElement(_Box.MotionBox, _extends({
    initial: {
      opacity: 0
    },
    animate: {
      opacity: 1
    },
    exit: {
      opacity: 0
    },
    boxShadow: 5,
    borderRadius: 8,
    border: "1px solid",
    borderColor: "grey.light",
    backgroundColor: "base.white",
    py: 2,
    maxWidth: maxWidth,
    "data-test": testId
  }, otherProps), children))));
};

exports.Dropdown = Dropdown;
