// @aircall/tractor/es/components/Select/components/Loading.js
import React from 'react';
import styled from 'styled-components';
import { Flex, Icon, SpinnerOutlined } from '../..';
import { Typography } from '../../Typography';
var LoadingContainer = styled(Flex).attrs(function () {
  return {
    alignItems: 'center',
    px: 's'
  };
}).withConfig({
  displayName: "Loading__LoadingContainer",
  componentId: "sc-xjrkin-0"
})(["height:", ";"], function (_ref) {
  var theme = _ref.theme,
      size = _ref.size;
  return theme.select.optionSizes[size].height;
});
export var Loading = function Loading(_ref2) {
  var loadingText = _ref2.loadingText,
      size = _ref2.size;
  return /*#__PURE__*/React.createElement(LoadingContainer, {
    "data-test": "select-loading",
    size: size,
    justifyContent: !loadingText ? 'center' : undefined
  }, /*#__PURE__*/React.createElement(Icon, {
    spin: true,
    component: SpinnerOutlined,
    color: "primary.base",
    size: "s"
  }), /*#__PURE__*/React.createElement(Typography, {
    ml: "xxs",
    variant: "body2"
  }, loadingText));
};
