// @aircall/tractor/es/components/Select/components/styled.js
import styled, { css } from 'styled-components';
import { Flex, FormItemStatus, Grid, TextFieldInput } from '../..';
export var DropdownContainer = styled(Flex).attrs(function () {
  return {
    flexDirection: 'column',
    backgroundColor: 'base.white',
    boxShadow: 3,
    borderRadius: 8,
    borderWidth: '1px',
    borderStyle: 'solid',
    borderColor: 'grey.base',
    minWidth: '200px'
  };
}).withConfig({
  displayName: "styled__DropdownContainer",
  componentId: "sc-1a4l9sz-0"
})(["overflow:hidden;"]);
export var CustomInput = styled(TextFieldInput).withConfig({
  displayName: "styled__CustomInput",
  componentId: "sc-1a4l9sz-1"
})(["border:none;", ";"], function (_ref) {
  var dropdownPosition = _ref.dropdownPosition,
      theme = _ref.theme;
  return dropdownPosition === 'bottom' ? css(["border-bottom:1px solid ", ";"], theme.colors.grey.base) : css(["border-top:1px solid ", ";"], theme.colors.grey.base);
});

var makeTriggerContainerStyle = function makeTriggerContainerStyle(_ref2) {
  var validationStatus = _ref2.validationStatus,
      theme = _ref2.theme,
      triggerSize = _ref2.triggerSize;
  return css(["height:", ";border-color:", ";"], theme.select.sizes[triggerSize].height, validationStatus === FormItemStatus.ERROR ? theme.colors.red.dark : theme.colors.grey.base);
};

var makeDisabledTriggerContainerStyle = function makeDisabledTriggerContainerStyle(_ref3) {
  var disabled = _ref3.disabled,
      theme = _ref3.theme;
  if (!disabled) return;
  return css(["pointer-events:none;background-color:", ";"], theme.colors.grey.light);
};

export var TriggerContainer = styled(Grid).attrs(function () {
  return {
    alignItems: 'center',
    borderWidth: '1px',
    borderStyle: 'solid',
    backgroundColor: 'base.white',
    cursor: 'pointer',
    borderRadius: 8,
    px: 's'
  };
}).withConfig({
  displayName: "styled__TriggerContainer",
  componentId: "sc-1a4l9sz-2"
})(["text-align:left;", " ", ""], makeTriggerContainerStyle, makeDisabledTriggerContainerStyle);
export var SelectGroupLabel = styled.div.withConfig({
  displayName: "styled__SelectGroupLabel",
  componentId: "sc-1a4l9sz-3"
})(["height:", ";"], function (_ref4) {
  var theme = _ref4.theme,
      size = _ref4.size;
  return theme.select.optionSizes[size];
});
