// @aircall/tractor/lib/components/Gauge/utils.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.createVolumeAnalyserData = exports.PollVolumeClass = void 0;
var FFT_SIZE = 32;

var createVolumeAnalyserData = function createVolumeAnalyserData(mediaStream) {
  var audioContext = new AudioContext();
  var mediaStreamSource = audioContext.createMediaStreamSource(mediaStream);
  var analyserNode = audioContext.createAnalyser();
  analyserNode.fftSize = FFT_SIZE;
  analyserNode.smoothingTimeConstant = 0.3;
  mediaStreamSource.connect(analyserNode);
  var analyserDataBuffer = new Uint8Array(analyserNode.frequencyBinCount);
  return {
    audioContext: audioContext,
    analyserNode: analyserNode,
    analyserDataBuffer: analyserDataBuffer,
    mediaStreamSource: mediaStreamSource
  };
};

exports.createVolumeAnalyserData = createVolumeAnalyserData;

var PollVolumeClass = function PollVolumeClass() {
  var _this = this;

  this.isPolling = true;

  this.pollVolume = function (analyserNode, analyserDataBuffer, setVolumeLevel) {
    if (!_this.isPolling) {
      return;
    }

    analyserNode.getByteFrequencyData(analyserDataBuffer);
    var volume = analyserDataBuffer.reduce(function (a, b) {
      return a + b;
    }, 0) / analyserDataBuffer.length;
    var volumeLvl = volume / 255;
    setVolumeLevel(volumeLvl);
    window.requestAnimationFrame(function () {
      return _this.pollVolume(analyserNode, analyserDataBuffer, setVolumeLevel);
    });
  };
};

exports.PollVolumeClass = PollVolumeClass;
