// @aircall/tractor/es/components/Flag/components/Abw.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgAbw(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#2C97DE"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.428 22h31.13c-.11.665-.24 1.333-.395 2H.835a27.05 27.05 0 01-.407-2zm1.101 4H30.47a10.672 10.672 0 01-1.196 2H2.725a10.674 10.674 0 01-1.196-2z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M9.013 8.013L8 5 6.993 7.993 4 9l3.013 1.013L8 13l.993-3.007L12 9l-2.987-.987z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M8.76 8.26L8 6l-.755 2.245L5 9l2.26.76L8 12l.745-2.255L11 9l-2.24-.74z",
    fill: "#E94B35"
  }));
}

export default SvgAbw;
