// @aircall/tractor/es/components/Flag/components/Ala.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgAla(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M12 .18v31.64c-2.279-.193-4.32-.54-5.657-1.05a8.008 8.008 0 01-1.01-.475V1.705a7.99 7.99 0 011.01-.476C7.679.72 9.72.373 12 .18z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.117 12.667h31.778c.088 1.152.116 2.277.102 3.35.014 1.055-.02 2.17-.114 3.316H.107a36.974 36.974 0 01-.104-3.35c-.014-1.055.02-2.17.114-3.316z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10.666.314v31.372c-1.564-.184-2.957-.45-4-.8V1.113c1.043-.35 2.436-.615 4-.799z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.033 14h31.938c.027.688.034 1.362.026 2.016.008.642-.001 1.305-.03 1.984H.029a38.365 38.365 0 01-.026-2.017c-.008-.64.001-1.304.03-1.983z",
    fill: "#E94B35"
  }));
}

export default SvgAla;
