// @aircall/tractor/es/components/Flag/components/Bra.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgBra(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 6.667L2.666 16 16 25.333 29.333 16 16 6.667z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M22 16a6 6 0 10-12 0 6 6 0 0012 0z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M21.96 16.697c-.045.39-.128.77-.244 1.134a11.794 11.794 0 00-8.22-3.322c-1.188 0-2.334.175-3.415.5.065-.387.166-.761.3-1.12.998-.248 2.04-.38 3.114-.38 3.243 0 6.206 1.204 8.465 3.188z",
    fill: "#fff"
  }));
}

export default SvgBra;
