// @aircall/tractor/es/components/Flag/components/Che.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgChe(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M9.015 14h13.973c.011.688.014 1.362.01 2.016.004.642 0 1.305-.013 1.984H9.013a87.583 87.583 0 01-.011-2.017c-.004-.64 0-1.304.013-1.983z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M14 22.985V9.012a89.06 89.06 0 012.016-.01c.642-.004 1.305 0 1.984.013v13.972c-.689.012-1.363.015-2.017.012-.64.003-1.304 0-1.983-.014z",
    fill: "#fff"
  }));
}

export default SvgChe;
