// @aircall/tractor/es/components/Flag/components/Cok.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgCok(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M24 10.685l-.669.493.236-.825-.65-.521.816-.018L24 9l.268.814.815.018-.65.52.236.826-.669-.493zm2.18.466l-.811.178.55-.658-.38-.74.751.316.576-.635-.087.853.738.347-.806.212-.12.85-.41-.723zm.99 1.15l.813.168.082.827.455-.727.822.134-.532-.618.426-.743-.784.346-.559-.595.048.832-.771.377zm1.928 2.11l-.675-.484.857-.03.295-.78.268.77.858.003-.692.506.235.781-.696-.457-.712.48.262-.789zm-.187 1.513l.42.717-.56.614.845-.148.45.7.103-.809.838-.18-.782-.352.068-.813-.586.592-.796-.321zm-.269 2.85l-.092-.826.597.617.776-.303-.393.715.572.639-.84-.175-.422.697-.126-.823-.833-.208.761-.334zm-1.24.874l-.251.792-.831-.005.676.528-.22.804.67-.465.695.501-.262-.816.65-.493-.832-.04-.294-.806zm-2.287 1.704l.552-.621-.06.856.744.374-.793.187-.093.852-.431-.74-.801.151.527-.644-.402-.758.757.343zm-1.472-.343l-.757.343-.552-.621.059.856-.744.374.794.187.092.852.432-.74.8.151-.527-.644.403-.758zm-2.793-.57l.83-.004-.675.528.22.804-.67-.465-.696.501.262-.816-.649-.493.832-.04.294-.806.252.792zm-.73-1.332l-.762-.334.092-.825-.596.617-.776-.303.392.715-.571.639.839-.175.423.697.126-.823.832-.208zm-1.45-2.466l.56.614-.846-.148-.45.7-.103-.809-.837-.18.781-.352-.067-.813.586.592.795-.321-.42.717zm.494-1.441l-.261-.789.675-.484-.858-.03-.294-.78-.269.77-.857.003.691.506-.235.781.696-.457.712.48zm.853-2.73l-.082.826-.455-.727-.822.134.533-.618-.426-.743.784.346.558-.595-.047.832.77.377-.813.167zm1.392-.597l.411-.722.812.178-.552-.658.382-.74-.752.316-.575-.635.086.853-.737.347.805.212.12.85z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 0v16H.003v-.017c-.033-2.492.2-5.316.863-8.108 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M4.288 2.402L8 6.114 14.065.05A56.78 56.78 0 0116 0v1.886L9.886 8 16 14.114V16h-1.886L8 9.886 1.886 16H.003v-.017c-.008-.614 0-1.249.027-1.898L6.114 8 2.467 4.353a9.023 9.023 0 011.821-1.95z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M14.555.031C15.035.015 15.518.004 16 0L8.472 7.529l-.708-.708 6.791-6.79zM3.308 3.307l4.22 4.222-.706.707-4.157-4.157c.194-.26.408-.52.643-.772zM.003 16v-.003L7.53 8.47l.707.708L1.414 16H.004zm8.469-7.529l.707-.707L16 14.586V16L8.472 8.471z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10 .399V16H6V1.372c.112-.05.226-.097.342-.143C7.272.875 8.544.6 10 .4z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M1.529 6H16v4H.442a30.35 30.35 0 01.424-2.125s.169-.803.663-1.875z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M9.334.498V16H6.667V1.114C7.403.867 8.312.662 9.334.498z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M1.246 6.667H16v2.666H.558c.09-.485.193-.972.308-1.458 0 0 .102-.487.38-1.208z",
    fill: "#E94B35"
  }));
}

export default SvgCok;
