// @aircall/tractor/es/components/Flag/components/Eth.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgEth(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.683 10.667H.34c.131-.925.305-1.86.526-2.792 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.22.916.39 1.84.516 2.758z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.34 10.667h31.343c.254 1.844.336 3.663.314 5.35a34.717 34.717 0 01-.337 5.316H.325a34.916 34.916 0 01-.322-5.35c-.022-1.66.074-3.469.337-5.316z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.325 21.333H31.66a30.81 30.81 0 01-.527 2.791s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.232-.93-.41-1.865-.54-2.79z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 23.333a7.333 7.333 0 100-14.666 7.333 7.333 0 000 14.666z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M12.28 21.267l3.743-2.605 3.743 2.605-1.32-4.365 3.633-2.755-4.56-.092-1.496-4.307-1.497 4.307-4.56.092 3.634 2.755-1.32 4.365zm6.292-1.643l-2.55-1.774-2.548 1.774.899-2.973-2.475-1.876 3.105-.063 1.02-2.934 1.02 2.934 3.105.063-2.475 1.876.9 2.973z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M13.6 16.902l2.423 1.76 2.422-1.76-.925-2.848h-2.995l-.925 2.848zm4.072-.25l-1.65 1.198-1.65-1.199.63-1.94h2.04l.63 1.94zM15.633 22.6h.667v-2.333h-.667V22.6zm6.254-4.06l.206-.633-2.22-.721-.205.634 2.219.72zm-1.928-7.201l-.54-.392-1.37 1.888.538.391 1.372-1.887zm-7.445-.392l-.54.392 1.372 1.887.54-.391-1.372-1.888zm-2.674 6.96l.206.634 2.22-.721-.207-.634-2.219.72z",
    fill: "#F2C500"
  }));
}

export default SvgEth;
