// @aircall/tractor/es/components/Flag/components/Ggy.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgGgy(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M18 .053V31.95a60.81 60.81 0 01-2 .05 56.666 56.666 0 01-2-.053V.053c.662-.03 1.332-.047 2-.053.668.006 1.338.023 2 .053z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.033 14h31.938c.027.688.034 1.362.026 2.016.008.642-.001 1.305-.03 1.984H.029a38.365 38.365 0 01-.026-2.017c-.008-.64.001-1.304.03-1.983z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M8 17.66l1.007-.83h6.163v6.163L14.34 24h3.32l-.83-1.007V16.83h6.163l1.007.83v-3.32l-1.007.83H16.83V9.007L17.66 8h-3.32l.83 1.007v6.163H9.007L8 14.34v3.32z",
    fill: "#F2C500"
  }));
}

export default SvgGgy;
