// @aircall/tractor/es/components/Flag/components/Isl.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgIsl(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M13.333.086v31.827C11.36 31.801 9.49 31.581 8 31.25V.75C9.49.418 11.36.2 13.333.086z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.931 18.667H.061c-.053-.92-.069-1.82-.058-2.684-.01-.85.009-1.737.066-2.65h31.87c.053.92.069 1.82.058 2.683.01.85-.009 1.738-.066 2.65z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M12 .18v31.64a36.87 36.87 0 01-2.666-.318V.498A36.3 36.3 0 0112 .18z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.99 17.333H.008c-.01-.458-.011-.908-.006-1.35-.005-.43-.003-.869.008-1.316h31.98c.01.457.011.908.006 1.35.005.43.003.869-.008 1.316z",
    fill: "#E94B35"
  }));
}

export default SvgIsl;
