// @aircall/tractor/es/components/Flag/components/Ken.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgKen(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.34 10.667h31.343c.254 1.844.336 3.663.314 5.35a34.717 34.717 0 01-.337 5.316H.325a34.914 34.914 0 01-.322-5.35c-.022-1.66.074-3.469.337-5.316z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.178 12h31.66c.133 1.384.175 2.738.159 4.016A35.014 35.014 0 0131.822 20H.166a36.282 36.282 0 01-.163-4.017A35.024 35.024 0 01.178 12z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.325 21.333H31.66a30.81 30.81 0 01-.527 2.791s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.232-.93-.41-1.865-.54-2.79z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.683 10.667H.34c.131-.925.305-1.86.526-2.792 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.22.916.39 1.84.516 2.758z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 25.333c1.244 0 4.667-4.178 4.667-9.333S17.244 6.667 16 6.667c-1.244 0-4.666 4.178-4.666 9.333s3.422 9.333 4.666 9.333z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M19.89 20.299c-.742-.971-1.223-2.535-1.223-4.299 0-1.764.481-3.328 1.224-4.299A12.65 12.65 0 0120.667 16c0 1.55-.31 3.012-.776 4.299zM12.11 11.7c.742.971 1.224 2.535 1.224 4.299 0 1.764-.482 3.328-1.224 4.299A12.65 12.65 0 0111.334 16c0-1.55.309-3.012.776-4.299z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 17.333a1.333 1.333 0 100-2.666 1.333 1.333 0 000 2.666z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 14.667s1.333-1.791 1.333-4c0-2.21-1.333-4-1.333-4s-1.334 1.79-1.334 4c0 2.209 1.334 4 1.334 4zm0 10.666s1.333-1.79 1.333-4c0-2.209-1.333-4-1.333-4s-1.334 1.791-1.334 4c0 2.21 1.334 4 1.334 4z",
    fill: "#fff"
  }));
}

export default SvgKen;
