// @aircall/tractor/es/components/Flag/components/Mkd.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgMkd(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.845 12.073L16 16 28.713 3.327c1.927 2.065 2.454 4.582 2.454 4.582.334 1.386.55 2.79.678 4.164zm-3.134 16.599L16 16l15.83 3.923a31.778 31.778 0 01-.697 4.201s-.52 2.49-2.422 4.548zM3.312 3.302L16 15.95.175 12.028a31.72 31.72 0 01.691-4.153s.525-2.51 2.446-4.573zM.158 19.926L16 16 3.288 28.672C1.387 26.613.866 24.124.866 24.124a28.651 28.651 0 01-.708-4.198z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M7.218.945c1.59-.443 3.787-.727 6.13-.86L16 16 7.218.945zm6.13 30.969c-2.342-.132-4.54-.416-6.13-.86L15.988 16l-2.64 15.914zM18.652.086c2.33.133 4.518.418 6.122.873L16 16 18.652.086zm6.125 30.98c-1.592.454-3.791.727-6.135.853L16 16l8.777 15.066z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M20.667 16a4.667 4.667 0 10-9.334 0 4.667 4.667 0 009.334 0z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M20 16a4 4 0 10-8 0 4 4 0 008 0z",
    fill: "#F2C500"
  }));
}

export default SvgMkd;
