// @aircall/tractor/es/components/Flag/components/Prt.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgPrt(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10.667 31.686V.314A50.525 50.525 0 0116 0c3.75.033 7.567.432 9.691 1.263 4.447 1.728 5.476 6.646 5.476 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.863 8.108 0 0-1.029 4.918-5.476 6.646-2.09.83-5.908 1.196-9.658 1.23a50.558 50.558 0 01-5.333-.314z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10.667.314v31.372c-1.733-.204-3.255-.508-4.325-.916C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.318.863-8.109 0 0 1.03-4.918 5.476-6.646C7.412.821 8.934.518 10.667.314z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10.667 22a6.667 6.667 0 110-13.333 6.667 6.667 0 010 13.333zm0-1.333a5.333 5.333 0 100-10.667 5.333 5.333 0 000 10.667z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M14.333 11.667V16A3.667 3.667 0 117 16v-4.333h7.333z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M7.333 12H14v4a3.333 3.333 0 01-6.667 0v-4z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M8.667 16a2 2 0 104 0v-2.667h-4V16z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#2C97DE",
    d: "M10.333 14H11v.667h-.667zm-1 1.333H10V16h-.667zm2 0H12V16h-.667zm-1 1.334H11v.667h-.667zm0-1.334H11V16h-.667z"
  }));
}

export default SvgPrt;
