// @aircall/tractor/es/components/Flag/components/Pry.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgPry(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.34 10.667h31.343c.254 1.844.336 3.663.314 5.35a34.717 34.717 0 01-.337 5.316H.325a34.916 34.916 0 01-.322-5.35c-.022-1.66.074-3.469.337-5.316z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.325 21.333H31.66a30.81 30.81 0 01-.527 2.791s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.232-.93-.41-1.865-.54-2.79z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.683 10.667H.34c.131-.925.305-1.86.526-2.792 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.22.916.39 1.84.516 2.758z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M11.334 16a4.667 4.667 0 109.333 0 4.667 4.667 0 00-9.334 0zm9 0a4.333 4.333 0 11-8.667 0 4.333 4.333 0 018.667 0z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16.994 12.817a3.308 3.308 0 00-.593-.127.333.333 0 01.08-.662c.242.03.48.08.711.152a.333.333 0 01-.198.637zm-2.155.057c.189-.07.384-.123.585-.158a.333.333 0 00-.115-.656c-.24.041-.475.105-.702.19a.333.333 0 10.232.624zm-.799.43a3.36 3.36 0 00-.455.4.333.333 0 11-.484-.46c.168-.176.35-.337.547-.48a.333.333 0 01.393.54zm-1.236 1.747c.057-.194.133-.382.224-.562a.333.333 0 10-.594-.303c-.11.216-.2.443-.27.676a.333.333 0 10.64.19zm-.137.99a3.3 3.3 0 00.053.554.333.333 0 11-.656.118c-.04-.218-.06-.44-.063-.667v-.106a.333.333 0 11.666.01v.089zm.687 1.986a3.334 3.334 0 01-.324-.512.333.333 0 00-.593.303c.11.217.24.422.388.615a.333.333 0 00.53-.406zm.654.646c.163.121.336.227.517.317a.333.333 0 01-.296.598 4.007 4.007 0 01-.62-.38.333.333 0 01.4-.535zm2.07.67a.333.333 0 00-.078-.01c-.204 0-.406-.018-.604-.054a.333.333 0 00-.12.655c.238.044.48.066.724.066a.335.335 0 00.094-.013c.03.007.061.01.094.009.244-.012.485-.045.72-.1a.333.333 0 10-.15-.649 3.333 3.333 0 01-.601.083.334.334 0 00-.078.013zm0 0a.334.334 0 00.016.644.334.334 0 00-.015-.644zm1.758-6.126c.169.112.327.238.473.379a.333.333 0 10.462-.481 4.016 4.016 0 00-.568-.454.333.333 0 10-.367.556zm1.3 1.647a3.31 3.31 0 00-.258-.548.333.333 0 11.575-.338c.122.21.226.43.309.659a.333.333 0 01-.627.227zm.194.968a3.377 3.377 0 01-.026.617.333.333 0 10.66.09 4.082 4.082 0 00.031-.74.333.333 0 00-.666.033zm-.564 2.03c.113-.168.21-.346.292-.531a.333.333 0 11.61.266 4 4 0 01-.35.638.333.333 0 01-.552-.373zm-.614.684c-.155.13-.32.247-.496.348a.333.333 0 10.331.578c.211-.12.41-.26.596-.417a.333.333 0 00-.43-.51z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 16.873l-1.11.794.416-1.27-.972-.794h1.25l.416-1.27.417 1.27h1.25l-.972.794.416 1.27-1.11-.794z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M14.714 13.286a3 3 0 102.572 0 .333.333 0 10-.286.602 2.333 2.333 0 11-2 0 .333.333 0 10-.286-.602z",
    fill: "#1FCE6D"
  }));
}

export default SvgPry;
