// @aircall/tractor/es/components/Flag/components/Sgp.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgSgp(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.997 16H.003v-.017c-.033-2.492.2-5.316.863-8.108 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.662 2.752.862 5.57.83 8.091z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.003 16h31.994v.016c.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124.171 21.34-.029 18.521.003 16zM9.24 3.002C6.524 3.073 4.346 5.284 4.346 8s2.178 4.927 4.892 4.998a5.379 5.379 0 01-1.875.335C4.4 13.333 2 10.945 2 8c0-2.946 2.401-5.333 5.364-5.333.66 0 1.291.118 1.875.335z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M9.71 5.733l-.59.41.208-.686-.574-.433.72-.014.236-.677.237.677.72.014-.574.433.208.685-.59-.409zm-2.704 2l-.591.41.208-.686-.574-.433.72-.014.237-.677.236.677.72.014-.574.433.209.685-.591-.409zm1 3l-.591.41.208-.686-.574-.433.72-.014.237-.677.236.677.72.014-.574.433.209.685-.591-.409zm3.333 0l-.591.41.209-.686-.574-.433.72-.014.236-.677.237.677.72.014-.574.433.208.685-.59-.409zm1-3l-.591.41.209-.686-.574-.433.72-.014.236-.677.237.677.72.014-.574.433.208.685-.59-.409z",
    fill: "#fff"
  }));
}

export default SvgSgp;
