// @aircall/tractor/es/components/Flag/components/Tca.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgTca(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 0v16H.003v-.017c-.033-2.492.2-5.316.863-8.108 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M4.288 2.402L8 6.114 14.065.05A56.78 56.78 0 0116 0v1.886L9.886 8 16 14.114V16h-1.886L8 9.886 1.886 16H.003v-.017c-.008-.614 0-1.249.027-1.898L6.114 8 2.467 4.353a9.023 9.023 0 011.821-1.95z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M14.555.031C15.035.015 15.518.004 16 0L8.472 7.529l-.708-.708 6.791-6.79zM3.308 3.307l4.22 4.222-.706.707-4.157-4.157c.194-.26.408-.52.643-.772zM.003 16v-.003L7.53 8.47l.707.708L1.414 16H.004zm8.469-7.529l.707-.707L16 14.586V16L8.472 8.471z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M10 .399V16H6V1.372c.112-.05.226-.097.342-.143C7.272.876 8.544.6 10 .4z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M1.529 6H16v4H.442a30.35 30.35 0 01.424-2.125s.169-.803.663-1.875z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M9.334.498V16H6.667V1.114C7.403.867 8.312.662 9.334.498z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M1.246 6.667H16v2.666H.558c.09-.485.193-.972.308-1.458 0 0 .102-.487.38-1.208z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M30 14.958c0 4.178-1.795 7.385-5.706 9.307L24 24.41l-.294-.145C19.795 22.343 18 19.135 18 14.958v-3.625c0-.736.597-1.333 1.333-1.333h9.334c.736 0 1.333.597 1.333 1.333v3.625z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M29.333 14.958v-3.625a.667.667 0 00-.666-.666h-9.334a.667.667 0 00-.666.666v3.625c0 3.523 1.344 6.748 5.333 8.709 3.989-1.96 5.333-5.186 5.333-8.709z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M26.667 16c.736 0 1.333-.895 1.333-2s-.597-2-1.333-2c-.737 0-1.334.895-1.334 2s.597 2 1.334 2z",
    fill: "#9C3900"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M24 21.333c1.105 0 2-.447 2-1 0-.552-.895-3.666-2-3.666s-2 3.114-2 3.666c0 .553.895 1 2 1z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M21.346 15.964c.54-.115.792-1.084.563-2.165-.23-1.08-.854-1.863-1.394-1.748-.54.115-.792 1.084-.563 2.164.23 1.08.854 1.863 1.394 1.749z",
    fill: "#D9903C"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M21.181 15.964c-.54-.115-.792-1.084-.562-2.165.23-1.08.854-1.863 1.394-1.748.54.115.792 1.084.562 2.164-.23 1.08-.853 1.863-1.394 1.749z",
    fill: "#FF9EE1"
  }));
}

export default SvgTca;
