// @aircall/tractor/es/components/Flag/components/Tjk.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgTjk(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.34 10.667h31.343c.254 1.844.336 3.663.314 5.35a34.717 34.717 0 01-.337 5.316H.325a34.916 34.916 0 01-.322-5.35c-.022-1.66.074-3.469.337-5.316z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.325 21.333H31.66a30.81 30.81 0 01-.527 2.791s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.232-.93-.41-1.865-.54-2.79z",
    fill: "#1AAF5D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.683 10.667H.34c.131-.925.305-1.86.526-2.792 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.22.916.39 1.84.516 2.758z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M11.445 17.305l-.588.41.207-.686-.57-.433.716-.014.235-.677.235.677.716.014-.57.433.207.685-.588-.409zm.615-2.346l-.587.409.207-.686-.57-.432.715-.015.235-.676.236.676.716.015-.571.432.207.686-.588-.41zm1.63-1.758l-.588.409.208-.685-.57-.433.715-.015.235-.676.235.676.716.015-.57.433.207.685-.588-.409zm2.311-.595l-.587.41.207-.686-.57-.433.715-.014.235-.677.235.677.716.014-.57.433.207.685-.588-.409zm2.191.63l-.588.409.207-.685-.57-.433.716-.015.235-.676.235.676.716.015-.57.433.207.685-.588-.409zm1.605 1.722l-.588.41.208-.686-.57-.433.715-.014.235-.677.235.677.716.014-.57.433.207.685-.588-.409zm.623 2.382l-.587.41.207-.686-.57-.433.716-.014.235-.677.235.677.716.014-.57.433.207.686-.588-.41z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M13.14 20.636A6.712 6.712 0 0116 20a6.834 6.834 0 012.861.636.333.333 0 10.278-.606A7.5 7.5 0 0016 19.333a7.38 7.38 0 00-3.14.698.333.333 0 00.279.605zM16 18.082c.589.4 1.213.509 1.91.248.517.546 1.176.653 1.901.304a.333.333 0 00-.289-.601c-.52.25-.902.157-1.25-.306a.333.333 0 00-.412-.1c-.627.303-1.121.207-1.65-.22a.333.333 0 00-.42 0c-.528.427-1.022.523-1.65.22a.334.334 0 00-.412.1c-.347.463-.73.557-1.25.306a.333.333 0 10-.29.6c.726.35 1.385.242 1.903-.302.696.26 1.32.15 1.91-.25z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M15.667 15.09a.664.664 0 00-.663-.003 1 1 0 111.993 0 .664.664 0 00-.664.002V18h-.666v-2.91z",
    fill: "#F2C500"
  }));
}

export default SvgTjk;
