// @aircall/tractor/es/components/Flag/components/Tls.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgTls(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M3.436 3.172L17.333 16 3.436 28.828c-2.022-2.087-2.57-4.704-2.57-4.704-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.318.863-8.109 0 0 .548-2.617 2.57-4.703z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M2.975 3.686L12.91 16 2.975 28.314c-1.646-1.978-2.109-4.19-2.109-4.19-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.318.863-8.109 0 0 .463-2.21 2.109-4.189z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M6.133 17.386l-1.218 2.592-.652-2.789-2.842-.357 2.45-1.483-.538-2.813L5.5 14.409l2.51-1.382-1.112 2.64 2.09 1.96-2.855-.241z",
    fill: "#fff"
  }));
}

export default SvgTls;
