// @aircall/tractor/es/components/Flag/components/Tto.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgTto(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#000"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M3.651 2.957l19.57 28.465c-2.055.377-4.654.555-7.221.578-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 .586-2.8 2.785-4.918zm24.696 26.086L8.787.592C10.84.216 13.435.022 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.584 2.427.808 4.906.831 7.187v.788l-.002.132.002.134v.657c-.022 2.29-.27 4.817-.865 7.317 0 0-.586 2.802-2.786 4.92z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M2.738 3.982l19.019 27.664c-1.76.225-3.768.336-5.757.354-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.318.863-8.109 0 0 .418-1.997 1.872-3.893zm26.524 24.035L10.25.365C12.008.138 14.013.018 16 0c3.75.033 7.567.432 9.69 1.263 4.447 1.728 5.477 6.646 5.477 6.646.663 2.758.862 5.582.83 8.107.032 2.493-.2 5.317-.864 8.108 0 0-.418 1.997-1.871 3.893z",
    fill: "#E94B35"
  }));
}

export default SvgTto;
