// @aircall/tractor/es/components/Flag/components/Umi.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgUmi(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.997 16H.003v-.017c-.033-2.492.2-5.316.863-8.108 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.662 2.752.862 5.57.83 8.091z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M16 22a8.667 8.667 0 100-17.333A8.667 8.667 0 0016 22z",
    fill: "#F2C500"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.003 16h31.994v.016c.033 2.493-.2 5.317-.864 8.108 0 0-.415 1.985-1.858 3.876H2.725C1.282 26.11.866 24.124.866 24.124.171 21.34-.029 18.521.003 16z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M2.725 28h26.55c-.828 1.085-1.996 2.14-3.618 2.77-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23-1.622-.63-2.789-1.684-3.617-2.77z",
    fill: "#F2C500"
  }));
}

export default SvgUmi;
