// @aircall/tractor/es/components/Flag/components/Uzb.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgUzb(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.34 10.667h31.343c.254 1.844.336 3.663.314 5.35a34.717 34.717 0 01-.337 5.316H.325a34.914 34.914 0 01-.322-5.35c-.022-1.66.074-3.469.337-5.316z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M31.683 10.667H.34c.131-.925.305-1.86.526-2.792 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263 4.448 1.728 5.477 6.646 5.477 6.646.22.916.39 1.84.516 2.758z",
    fill: "#4BB7C0"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M8.123 8.842l-.464.335.164-.56-.45-.355.564-.012.186-.553.186.553.565.012-.45.354.163.561-.464-.335zm2.105 0l-.464.335.164-.56-.45-.355.565-.012.185-.553.186.553.565.012-.45.354.163.561-.464-.335zm2.105 0l-.464.335.164-.56-.45-.355.565-.012.185-.553.186.553.565.012-.45.354.163.561-.464-.335zm2.106 0l-.464.335.164-.56-.451-.355.565-.012.186-.553.185.553.566.012-.45.354.163.561-.464-.335zm2.105 0l-.464.335.164-.56-.451-.355.565-.012.186-.553.185.553.566.012-.45.354.163.561-.464-.335zM10.228 6.66l-.464.335.164-.56-.45-.354.565-.012.185-.554.186.554.565.011-.45.354.163.561-.464-.334zm2.105 0l-.464.335.164-.56-.45-.354.565-.012.185-.554.186.554.565.011-.45.354.163.561-.464-.334zm2.106 0l-.464.335.164-.56-.451-.354.565-.012.186-.554.185.554.566.011-.45.354.163.561-.464-.334zm2.105 0l-.464.335.164-.56-.451-.354.565-.012.186-.554.185.554.566.011-.45.354.163.561-.464-.334zm-4.211-2.181l-.464.334.164-.56-.45-.354.565-.012.185-.554.186.554.565.012-.45.354.163.56-.464-.334zm2.106 0l-.464.334.164-.56-.451-.354.565-.012.186-.554.185.554.566.012-.45.354.163.56-.464-.334zm2.105 0l-.464.334.164-.56-.451-.354.565-.012.186-.554.185.554.566.012-.45.354.163.56-.464-.334zM7.333 2.713C5.713 2.977 4.476 4.348 4.476 6s1.236 3.023 2.857 3.287c-.185.03-.376.046-.57.046-1.894 0-3.43-1.492-3.43-3.333 0-1.84 1.536-3.333 3.43-3.333.194 0 .385.015.57.046z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.325 21.333H31.66a30.81 30.81 0 01-.527 2.791s-1.028 4.918-5.476 6.646C23.567 31.6 19.75 31.966 16 32c-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.232-.93-.41-1.865-.54-2.79z",
    fill: "#1FCE6D"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M.178 12h31.66c.133 1.384.175 2.738.159 4.016A35.014 35.014 0 0131.822 20H.166a36.282 36.282 0 01-.163-4.017A35.024 35.024 0 01.178 12z",
    fill: "#fff"
  }));
}

export default SvgUzb;
