// @aircall/tractor/es/components/Flag/components/Wlf.js
import _extends from "@babel/runtime/helpers/esm/extends";
import * as React from 'react';

function SvgWlf(props) {
  return /*#__PURE__*/React.createElement("svg", _extends({
    width: "24px",
    height: "24px",
    viewBox: "0 0 32 32",
    fill: "none"
  }, props), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M25.69 1.263c4.448 1.728 5.477 6.646 5.477 6.646.664 2.758.863 5.582.83 8.107.033 2.493-.2 5.317-.864 8.108 0 0-1.028 4.918-5.476 6.646-2.09.83-5.907 1.196-9.657 1.23-3.75-.034-7.567-.432-9.658-1.23C1.895 29.042.866 24.124.866 24.124c-.697-2.79-.896-5.615-.863-8.14-.033-2.493.2-5.317.863-8.109 0 0 1.03-4.918 5.476-6.646C8.433.432 12.25.033 16 0c3.75.033 7.567.432 9.69 1.263z",
    fill: "#E94B35"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M14.666 17.333l6 3.334-3.333-6h8l-3.333 6 6-3.334v8L22 22l3.333 6h-8l3.334-6-6 3.333v-8zM.34 10.667c.131-.925.305-1.86.526-2.792 0 0 1.03-4.918 5.476-6.646C8.185.527 11.37.133 14.667.027v10.64H.34z",
    fill: "#fff"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M4.667 2.122V10H.442a30.35 30.35 0 01.424-2.125s.761-3.635 3.8-5.753z",
    fill: "#227FBB"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "evenodd",
    clipRule: "evenodd",
    d: "M14 .053V10H9.334V.498C10.733.274 12.34.126 14 .053z",
    fill: "#E94B35"
  }));
}

export default SvgWlf;
