// @aircall/tractor/lib/components/Skeleton/Skeleton.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Skeleton = void 0;

var _react = _interopRequireDefault(require("react"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _Box = require("../Box");

var _utils = require("./utils");

var _excluded = ["shape", "animation"];

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

var SkeletonComponent = (0, _styledComponents["default"])(_Box.Box).withConfig({
  shouldForwardProp: _utils.shouldForwardProp
}).withConfig({
  displayName: "Skeleton__SkeletonComponent",
  componentId: "sc-1ol6tq0-0"
})(["", " ", " position:relative;overflow:hidden;"], _utils.makeSkeletonShape, _utils.makeSkeletonAnimation);

var Skeleton = function Skeleton(_ref) {
  var _ref$shape = _ref.shape,
      shape = _ref$shape === void 0 ? 'squareRounded' : _ref$shape,
      _ref$animation = _ref.animation,
      animation = _ref$animation === void 0 ? 'wave' : _ref$animation,
      rest = _objectWithoutPropertiesLoose(_ref, _excluded);

  return /*#__PURE__*/_react["default"].createElement(SkeletonComponent, _extends({
    bg: "grey.light",
    shape: shape,
    animation: animation
  }, rest));
};

exports.Skeleton = Skeleton;
