// @aircall/tractor/es/components/Toast/components.js
import styled, { keyframes, css } from 'styled-components';
import { Flex } from '../Flex';
import { Typography } from '../Typography';
import { makeToastVariant } from './utils';
export var ToastContainer = styled(Flex).withConfig({
  displayName: "components__ToastContainer",
  componentId: "sc-19b1rha-0"
})(function (_ref) {
  var theme = _ref.theme,
      takeContainerWidth = _ref.takeContainerWidth;
  return css(["align-items:center;justify-content:space-between;border-radius:", ";", " margin-bottom:", "px;padding:", "px ", "px;margin-bottom:", "px;user-select:none;min-height:40px;width:", ";max-width:", ";"], theme.radii[8], makeToastVariant, theme.space[2], theme.space[2], theme.space[4], theme.space[2], takeContainerWidth ? '100%' : '344px', takeContainerWidth ? 'auto' : 'calc(100vw - 24px * 2)');
});
var progress = keyframes(["from{stroke-dashoffset:-82px;}to{stroke-dashoffset:-164px;}"]);
export var SVGContainer = styled.svg.withConfig({
  displayName: "components__SVGContainer",
  componentId: "sc-19b1rha-1"
})(["overflow:visible;position:absolute;transform:rotate(-90deg);"]);
export var IndicatorCircle = styled.circle.withConfig({
  displayName: "components__IndicatorCircle",
  componentId: "sc-19b1rha-2"
})(["fill:transparent;stroke-dasharray:82px;stroke-width:1px;stroke:currentColor;opacity:0.4;stroke-dashoffset:0;transition:stroke 0.2s ease-in-out;", ":hover &{opacity:1;}", ":active &{fill:rgb(255,255,255,0.1);}"], SVGContainer, SVGContainer);
export var ProgressCircle = styled.circle.withConfig({
  displayName: "components__ProgressCircle",
  componentId: "sc-19b1rha-3"
})(function (_ref2) {
  var dismissIn = _ref2.dismissIn;
  return css(["fill:transparent;stroke-dasharray:82px;stroke-width:1px;animation-delay:0.5s;animation-name:", ";animation-timing-function:linear;animation-fill-mode:forwards;animation-duration:", "ms;stroke:currentColor;stroke-dashoffset:", ";stroke-linecap:round;"], progress, dismissIn, dismissIn === 0 ? '-164px' : '-82px');
});
export var Action = styled(Typography).withConfig({
  displayName: "components__Action",
  componentId: "sc-19b1rha-4"
})(["", ""], function (_ref3) {
  var onClick = _ref3.onClick;
  return onClick && css(["cursor:pointer;&:hover{text-decoration:underline;}"]);
});
