// @aircall/tractor/lib/components/Tab/__stories__/styled.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.StyledTabMenuItem = exports.StyledTabMenu = exports.StyledTabItem = exports.StyledTabContainer = exports.StyledBox = void 0;

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _ = require("..");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var StyledTabContainer = (0, _styledComponents["default"])(_.Tab.Container).withConfig({
  displayName: "styled__StyledTabContainer",
  componentId: "sc-k1m6p1-0"
})(["min-height:500px;background:#fcfcfc;"]);
exports.StyledTabContainer = StyledTabContainer;
var StyledTabItem = (0, _styledComponents["default"])(_.Tab.Item).withConfig({
  displayName: "styled__StyledTabItem",
  componentId: "sc-k1m6p1-1"
})(["padding:", "px;background:transparent;"], function (_ref) {
  var theme = _ref.theme;
  return theme.space.xs;
});
exports.StyledTabItem = StyledTabItem;
var StyledTabMenu = (0, _styledComponents["default"])(_.Tab.Menu).withConfig({
  displayName: "styled__StyledTabMenu",
  componentId: "sc-k1m6p1-2"
})(["border-bottom:none;background:transparent;padding:", "px 0;> div > div{margin:0;}"], function (_ref2) {
  var theme = _ref2.theme;
  return theme.space.xs;
});
exports.StyledTabMenu = StyledTabMenu;
var StyledTabMenuItem = (0, _styledComponents["default"])(_.Tab.MenuItem).attrs({
  activeTabColor: 'primary.base'
}).withConfig({
  displayName: "styled__StyledTabMenuItem",
  componentId: "sc-k1m6p1-3"
})(["color:'#707479';padding:10px 0px 25px 0;&:after{border-radius:2px;}"]);
exports.StyledTabMenuItem = StyledTabMenuItem;

var StyledBox = _styledComponents["default"].div.withConfig({
  displayName: "styled__StyledBox",
  componentId: "sc-k1m6p1-4"
})(["box-shadow:", ";margin:", "px 0;background:white;height:50px;line-height:50px;border-radius:", ";"], function (_ref3) {
  var theme = _ref3.theme;
  return theme.shadows[1];
}, function (_ref4) {
  var theme = _ref4.theme;
  return theme.space.xs;
}, function (_ref5) {
  var theme = _ref5.theme;
  return theme.radii[8];
});

exports.StyledBox = StyledBox;
