// @aircall/tractor/lib/components/Spacer/Spacer.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = exports.Spacer = void 0;

var _react = _interopRequireWildcard(require("react"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _styledSystem = require("styled-system");

var _utils = require("../../utils");

var _Box = require("../Box");

var _Flex = require("../Flex");

var _Spacer = require("./Spacer.decl");

var _utils2 = require("./utils");

var _excluded = ["children", "alignItems", "justifyContent", "flexWrap", "space", "wrapSpace", "direction", "reverse", "fluid", "width", "itemsSized"];

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

var Container = (0, _styledComponents["default"])(_Flex.Flex).withConfig({
  shouldForwardProp: _utils2.shouldForwardProp,
  displayName: "Spacer__Container",
  componentId: "sc-1d5auvu-0"
})(function (_ref) {
  var theme = _ref.theme,
      direction = _ref.direction,
      reverse = _ref.reverse,
      flexWrap = _ref.flexWrap,
      wrapSpace = _ref.wrapSpace;
  var flexDirection = direction === _Spacer.SpacerDirection.VERTICAL ? 'column' : 'row';
  var styleProps = {
    verticalAlign: 'top'
  };
  styleProps['flexDirection'] = reverse ? flexDirection + "-reverse" : flexDirection; // Remove the size added by the last row

  if (flexWrap && direction === _Spacer.SpacerDirection.HORIZONTAL) {
    /**
     * Support responsive wrapSpace value.
     * wrapSpace={{ xs: 's', lg: 'xs' }}
     */
    var result = (0, _styledSystem.space)({
      mb: wrapSpace,
      theme: theme
    });
    styleProps = _extends({}, styleProps, (0, _utils.mapValues)(result, function (value) {
      if (typeof value === 'object') {
        var marginBottom = value.marginBottom;
        return {
          marginBottom: marginBottom * -1
        };
      }

      return value * -1;
    }));
  }

  return styleProps;
});
var Spacer = /*#__PURE__*/(0, _react.forwardRef)(function (_ref2, ref) {
  var children = _ref2.children,
      alignItems = _ref2.alignItems,
      justifyContent = _ref2.justifyContent,
      flexWrap = _ref2.flexWrap,
      _ref2$space = _ref2.space,
      space = _ref2$space === void 0 ? 'xxxs' : _ref2$space,
      _ref2$wrapSpace = _ref2.wrapSpace,
      wrapSpace = _ref2$wrapSpace === void 0 ? space : _ref2$wrapSpace,
      _ref2$direction = _ref2.direction,
      direction = _ref2$direction === void 0 ? 'horizontal' : _ref2$direction,
      _ref2$reverse = _ref2.reverse,
      reverse = _ref2$reverse === void 0 ? false : _ref2$reverse,
      fluid = _ref2.fluid,
      width = _ref2.width,
      _ref2$itemsSized = _ref2.itemsSized,
      itemsSized = _ref2$itemsSized === void 0 ? 'auto' : _ref2$itemsSized,
      otherStyleProps = _objectWithoutPropertiesLoose(_ref2, _excluded);

  var validChildren = _react["default"].Children.toArray(children).filter(Boolean);

  return /*#__PURE__*/_react["default"].createElement(Container, _extends({
    display: "inline-flex",
    ref: ref,
    direction: direction,
    alignItems: alignItems,
    justifyContent: justifyContent,
    flexWrap: flexWrap,
    width: width != null ? width : fluid ? '100%' : undefined,
    reverse: reverse,
    wrapSpace: wrapSpace
  }, otherStyleProps), validChildren.map(function (child, childIdx) {
    return /*#__PURE__*/_react["default"].createElement(_Box.Box, _extends({
      key: childIdx
    }, (0, _utils2.getChildStyle)({
      space: space,
      wrapSpace: wrapSpace,
      direction: direction,
      reverse: reverse,
      flexWrap: flexWrap,
      childIdx: childIdx,
      childrenCount: validChildren.length,
      itemsSized: itemsSized
    })), child);
  }));
});
exports.Spacer = Spacer;
var _default = Spacer;
exports["default"] = _default;
