// @aircall/tractor/lib/components/BoundaryAlignment/constants.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Position = exports.OPPOSITE_POSITION = exports.OPPOSITE_ANCHOR = exports.Anchor = void 0;

var _OPPOSITE_POSITION, _OPPOSITE_ANCHOR;

var Position = {
  RIGHT: 'right',
  LEFT: 'left',
  TOP: 'top',
  BOTTOM: 'bottom'
};
exports.Position = Position;
var OPPOSITE_POSITION = (_OPPOSITE_POSITION = {}, _OPPOSITE_POSITION[Position.RIGHT] = Position.LEFT, _OPPOSITE_POSITION[Position.LEFT] = Position.RIGHT, _OPPOSITE_POSITION[Position.TOP] = Position.BOTTOM, _OPPOSITE_POSITION[Position.BOTTOM] = Position.TOP, _OPPOSITE_POSITION);
exports.OPPOSITE_POSITION = OPPOSITE_POSITION;
var Anchor = {
  START: 'start',
  CENTER: 'center',
  END: 'end'
};
exports.Anchor = Anchor;
var OPPOSITE_ANCHOR = (_OPPOSITE_ANCHOR = {}, _OPPOSITE_ANCHOR[Anchor.START] = Anchor.END, _OPPOSITE_ANCHOR[Anchor.CENTER] = Anchor.CENTER, _OPPOSITE_ANCHOR[Anchor.END] = Anchor.START, _OPPOSITE_ANCHOR);
exports.OPPOSITE_ANCHOR = OPPOSITE_ANCHOR;
