// @aircall/tractor/lib/components/Divider/Divider.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Divider = void 0;

var _react = _interopRequireDefault(require("react"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _Box = require("../Box");

var _Divider = require("./Divider.decl");

var _utils = require("./utils");

var _excluded = ["size"];

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

var Divider = (0, _styledComponents["default"])(function (_ref) {
  var size = _ref.size,
      otherProps = _objectWithoutPropertiesLoose(_ref, _excluded);

  return /*#__PURE__*/_react["default"].createElement(_Box.Box, otherProps);
}).withConfig({
  shouldForwardProp: _utils.shouldForwardProp
}).withConfig({
  displayName: "Divider",
  componentId: "sc-th8c6m-0"
})(["", ""], _utils.makeDividerSize);
exports.Divider = Divider;
Divider.defaultProps = {
  size: _Divider.DividerSize.SMALL,
  orientation: _Divider.DividerOrientation.VERTICAL,
  bg: 'grey.light'
};
