// @aircall/tractor/es/components/Tab/__stories__/styled.js
import styled from 'styled-components';
import { Tab } from '..';
export var StyledTabContainer = styled(Tab.Container).withConfig({
  displayName: "styled__StyledTabContainer",
  componentId: "sc-k1m6p1-0"
})(["min-height:500px;background:#fcfcfc;"]);
export var StyledTabItem = styled(Tab.Item).withConfig({
  displayName: "styled__StyledTabItem",
  componentId: "sc-k1m6p1-1"
})(["padding:", "px;background:transparent;"], function (_ref) {
  var theme = _ref.theme;
  return theme.space.xs;
});
export var StyledTabMenu = styled(Tab.Menu).withConfig({
  displayName: "styled__StyledTabMenu",
  componentId: "sc-k1m6p1-2"
})(["border-bottom:none;background:transparent;padding:", "px 0;> div > div{margin:0;}"], function (_ref2) {
  var theme = _ref2.theme;
  return theme.space.xs;
});
export var StyledTabMenuItem = styled(Tab.MenuItem).attrs({
  activeTabColor: 'primary.base'
}).withConfig({
  displayName: "styled__StyledTabMenuItem",
  componentId: "sc-k1m6p1-3"
})(["color:'#707479';padding:10px 0px 25px 0;&:after{border-radius:2px;}"]);
export var StyledBox = styled.div.withConfig({
  displayName: "styled__StyledBox",
  componentId: "sc-k1m6p1-4"
})(["box-shadow:", ";margin:", "px 0;background:white;height:50px;line-height:50px;border-radius:", ";"], function (_ref3) {
  var theme = _ref3.theme;
  return theme.shadows[1];
}, function (_ref4) {
  var theme = _ref4.theme;
  return theme.space.xs;
}, function (_ref5) {
  var theme = _ref5.theme;
  return theme.radii[8];
});
