// @aircall/tractor/lib/components/Select/components/Loading.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.Loading = void 0;

var _react = _interopRequireDefault(require("react"));

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _ = require("../..");

var _Typography = require("../../Typography");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var LoadingContainer = (0, _styledComponents["default"])(_.Flex).attrs(function () {
  return {
    alignItems: 'center',
    px: 's'
  };
}).withConfig({
  displayName: "Loading__LoadingContainer",
  componentId: "sc-xjrkin-0"
})(["height:", ";"], function (_ref) {
  var theme = _ref.theme,
      size = _ref.size;
  return theme.select.optionSizes[size].height;
});

var Loading = function Loading(_ref2) {
  var loadingText = _ref2.loadingText,
      size = _ref2.size;
  return /*#__PURE__*/_react["default"].createElement(LoadingContainer, {
    "data-test": "select-loading",
    size: size,
    justifyContent: !loadingText ? 'center' : undefined
  }, /*#__PURE__*/_react["default"].createElement(_.Icon, {
    spin: true,
    component: _.SpinnerOutlined,
    color: "primary.base",
    size: "s"
  }), /*#__PURE__*/_react["default"].createElement(_Typography.Typography, {
    ml: "xxs",
    variant: "body2"
  }, loadingText));
};

exports.Loading = Loading;
