// @aircall/tractor/lib/components/Select/components/styled.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.TriggerContainer = exports.SelectGroupLabel = exports.DropdownContainer = exports.CustomInput = void 0;

var _styledComponents = _interopRequireWildcard(require("styled-components"));

var _ = require("../..");

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

var DropdownContainer = (0, _styledComponents["default"])(_.Flex).attrs(function () {
  return {
    flexDirection: 'column',
    backgroundColor: 'base.white',
    boxShadow: 3,
    borderRadius: 8,
    borderWidth: '1px',
    borderStyle: 'solid',
    borderColor: 'grey.base',
    minWidth: '200px'
  };
}).withConfig({
  displayName: "styled__DropdownContainer",
  componentId: "sc-1a4l9sz-0"
})(["overflow:hidden;"]);
exports.DropdownContainer = DropdownContainer;
var CustomInput = (0, _styledComponents["default"])(_.TextFieldInput).withConfig({
  displayName: "styled__CustomInput",
  componentId: "sc-1a4l9sz-1"
})(["border:none;", ";"], function (_ref) {
  var dropdownPosition = _ref.dropdownPosition,
      theme = _ref.theme;
  return dropdownPosition === 'bottom' ? (0, _styledComponents.css)(["border-bottom:1px solid ", ";"], theme.colors.grey.base) : (0, _styledComponents.css)(["border-top:1px solid ", ";"], theme.colors.grey.base);
});
exports.CustomInput = CustomInput;

var makeTriggerContainerStyle = function makeTriggerContainerStyle(_ref2) {
  var validationStatus = _ref2.validationStatus,
      theme = _ref2.theme,
      triggerSize = _ref2.triggerSize;
  return (0, _styledComponents.css)(["height:", ";border-color:", ";"], theme.select.sizes[triggerSize].height, validationStatus === _.FormItemStatus.ERROR ? theme.colors.red.dark : theme.colors.grey.base);
};

var makeDisabledTriggerContainerStyle = function makeDisabledTriggerContainerStyle(_ref3) {
  var disabled = _ref3.disabled,
      theme = _ref3.theme;
  if (!disabled) return;
  return (0, _styledComponents.css)(["pointer-events:none;background-color:", ";"], theme.colors.grey.light);
};

var TriggerContainer = (0, _styledComponents["default"])(_.Grid).attrs(function () {
  return {
    alignItems: 'center',
    borderWidth: '1px',
    borderStyle: 'solid',
    backgroundColor: 'base.white',
    cursor: 'pointer',
    borderRadius: 8,
    px: 's'
  };
}).withConfig({
  displayName: "styled__TriggerContainer",
  componentId: "sc-1a4l9sz-2"
})(["text-align:left;", " ", ""], makeTriggerContainerStyle, makeDisabledTriggerContainerStyle);
exports.TriggerContainer = TriggerContainer;

var SelectGroupLabel = _styledComponents["default"].div.withConfig({
  displayName: "styled__SelectGroupLabel",
  componentId: "sc-1a4l9sz-3"
})(["height:", ";"], function (_ref4) {
  var theme = _ref4.theme,
      size = _ref4.size;
  return theme.select.optionSizes[size];
});

exports.SelectGroupLabel = SelectGroupLabel;
