// @aircall/tractor/lib/components/Grid/utils.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.shouldForwardProp = exports.grid = void 0;

var _styledSystem = require("styled-system");

var _styles = require("../../styles");

// Props that should not be forwarded to the HTML DOM elements
var shouldForwardProp = function shouldForwardProp(prop, defaultValidationFn) {
  return !['height', 'width', 'color', 'display'].includes(prop) && defaultValidationFn(prop);
};
/**
 * Only define the parent grid attributes because the other grid items are already defined on the Box component.
 */


exports.shouldForwardProp = shouldForwardProp;
var grid = (0, _styledSystem.system)({
  gridGap: {
    property: 'gridGap',
    scale: 'space',
    defaultScale: _styles.baseTheme.space
  },
  gridColumnGap: {
    property: 'gridColumnGap',
    scale: 'space',
    defaultScale: _styles.baseTheme.space
  },
  gridRowGap: {
    property: 'gridRowGap',
    scale: 'space',
    defaultScale: _styles.baseTheme.space
  },
  gridAutoFlow: true,
  gridAutoColumns: true,
  gridAutoRows: true,
  gridTemplateColumns: true,
  gridTemplateRows: true,
  gridTemplateAreas: true
});
exports.grid = grid;
