// @aircall/tractor/lib/components/Button/Button.decl.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "ButtonDirection", {
  enumerable: true,
  get: function get() {
    return _Spacer.SpacerDirection;
  }
});
exports.ButtonVariant = exports.ButtonSize = exports.ButtonMode = void 0;

var _common = require("../../declarations/common.decl");

var _Spacer = require("../Spacer");

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

var ButtonSize = _extends({
  X_SMALL: 'xSmall'
}, _common.ComponentSize);

exports.ButtonSize = ButtonSize;
var ButtonVariant;
exports.ButtonVariant = ButtonVariant;

(function (ButtonVariant) {
  ButtonVariant["PRIMARY"] = "primary";
  ButtonVariant["DESTRUCTIVE"] = "destructive";
  ButtonVariant["INSTRUCTIVE"] = "instructive";
  ButtonVariant["WARNING"] = "warning";
  ButtonVariant["GHOST"] = "ghost";
  ButtonVariant["DARK_GHOST"] = "darkGhost";
  ButtonVariant["BLACK"] = "black";
  ButtonVariant["WHITE"] = "white";
})(ButtonVariant || (exports.ButtonVariant = ButtonVariant = {}));

var ButtonMode;
exports.ButtonMode = ButtonMode;

(function (ButtonMode) {
  ButtonMode["FILL"] = "fill";
  ButtonMode["OUTLINE"] = "outline";
  ButtonMode["LINK"] = "link";
})(ButtonMode || (exports.ButtonMode = ButtonMode = {}));
