// @aircall/tractor/lib/components/Accordion/__stories__/styled.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.StyledHeader = exports.StyledContainer = exports.StyledBody = exports.BottomStickyHeader = void 0;

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _Accordion = require("../Accordion");

var _AccordionBody = require("../AccordionBody");

var _AccordionHeader = require("../AccordionHeader");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var StyledContainer = (0, _styledComponents["default"])(_Accordion.Accordion).withConfig({
  displayName: "styled__StyledContainer",
  componentId: "sc-ktblgi-0"
})(["height:100%;"]);
exports.StyledContainer = StyledContainer;
var StyledHeader = (0, _styledComponents["default"])(_AccordionHeader.AccordionHeader).withConfig({
  displayName: "styled__StyledHeader",
  componentId: "sc-ktblgi-1"
})(["color:blue;"]);
exports.StyledHeader = StyledHeader;
var BottomStickyHeader = (0, _styledComponents["default"])(_AccordionHeader.AccordionHeader).withConfig({
  displayName: "styled__BottomStickyHeader",
  componentId: "sc-ktblgi-2"
})(["margin-top:auto;"]);
exports.BottomStickyHeader = BottomStickyHeader;
var StyledBody = (0, _styledComponents["default"])(_AccordionBody.AccordionBody).withConfig({
  displayName: "styled__StyledBody",
  componentId: "sc-ktblgi-3"
})(["padding:5px;border:2px dotted purple;"]);
exports.StyledBody = StyledBody;
