// @aircall/tractor/lib/components/Box/Box.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.flexboxItem = exports["default"] = void 0;

var _styledComponents = _interopRequireDefault(require("styled-components"));

var _styledSystem = require("styled-system");

var _styles = require("../../styles");

var _utils = require("./utils");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

var flexboxItem = (0, _styledSystem.system)({
  flex: true,
  flexGrow: true,
  flexShrink: true,
  flexBasis: true,
  justifySelf: true,
  alignSelf: true,
  order: true
});
exports.flexboxItem = flexboxItem;
var gridItem = (0, _styledSystem.system)({
  gridArea: true,
  gridColumn: true,
  gridRow: true,
  gridColumnStart: true,
  gridColumnEnd: true,
  gridRowStart: true,
  gridRowEnd: true
});
var customProps = (0, _styledSystem.system)({
  cursor: true,
  userSelect: true
});
/**
 * A plain div which supports arbitrary margins and paddings, where necessary.
 * 🚨Use scarcely!
 */

var Box = _styledComponents["default"].div.withConfig({
  shouldForwardProp: _utils.shouldForwardProp
}).withConfig({
  displayName: "Box",
  componentId: "sc-14d2slk-0"
})(["", " ", " ", " ", " ", " ", " ", " ", " ", " ", ""], _styledSystem.space, _styledSystem.layout, _styledSystem.position, _styledSystem.shadow, _styledSystem.border, _styledSystem.color, flexboxItem, gridItem, _styledSystem.lineHeight, customProps);

Box.defaultProps = {
  theme: _styles.baseTheme,
  display: 'block'
};
var _default = Box;
exports["default"] = _default;
