// @aircall/tractor/lib/components/Toast/components.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.ToastContainer = exports.SVGContainer = exports.ProgressCircle = exports.IndicatorCircle = exports.Action = void 0;

var _styledComponents = _interopRequireWildcard(require("styled-components"));

var _Flex = require("../Flex");

var _Typography = require("../Typography");

var _utils = require("./utils");

function _getRequireWildcardCache(nodeInterop) { if (typeof WeakMap !== "function") return null; var cacheBabelInterop = new WeakMap(); var cacheNodeInterop = new WeakMap(); return (_getRequireWildcardCache = function _getRequireWildcardCache(nodeInterop) { return nodeInterop ? cacheNodeInterop : cacheBabelInterop; })(nodeInterop); }

function _interopRequireWildcard(obj, nodeInterop) { if (!nodeInterop && obj && obj.__esModule) { return obj; } if (obj === null || typeof obj !== "object" && typeof obj !== "function") { return { "default": obj }; } var cache = _getRequireWildcardCache(nodeInterop); if (cache && cache.has(obj)) { return cache.get(obj); } var newObj = {}; var hasPropertyDescriptor = Object.defineProperty && Object.getOwnPropertyDescriptor; for (var key in obj) { if (key !== "default" && Object.prototype.hasOwnProperty.call(obj, key)) { var desc = hasPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : null; if (desc && (desc.get || desc.set)) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } newObj["default"] = obj; if (cache) { cache.set(obj, newObj); } return newObj; }

var ToastContainer = (0, _styledComponents["default"])(_Flex.Flex).withConfig({
  displayName: "components__ToastContainer",
  componentId: "sc-19b1rha-0"
})(function (_ref) {
  var theme = _ref.theme,
      takeContainerWidth = _ref.takeContainerWidth;
  return (0, _styledComponents.css)(["align-items:center;justify-content:space-between;border-radius:", ";", " margin-bottom:", "px;padding:", "px ", "px;margin-bottom:", "px;user-select:none;min-height:40px;width:", ";max-width:", ";"], theme.radii[8], _utils.makeToastVariant, theme.space[2], theme.space[2], theme.space[4], theme.space[2], takeContainerWidth ? '100%' : '344px', takeContainerWidth ? 'auto' : 'calc(100vw - 24px * 2)');
});
exports.ToastContainer = ToastContainer;
var progress = (0, _styledComponents.keyframes)(["from{stroke-dashoffset:-82px;}to{stroke-dashoffset:-164px;}"]);

var SVGContainer = _styledComponents["default"].svg.withConfig({
  displayName: "components__SVGContainer",
  componentId: "sc-19b1rha-1"
})(["overflow:visible;position:absolute;transform:rotate(-90deg);"]);

exports.SVGContainer = SVGContainer;

var IndicatorCircle = _styledComponents["default"].circle.withConfig({
  displayName: "components__IndicatorCircle",
  componentId: "sc-19b1rha-2"
})(["fill:transparent;stroke-dasharray:82px;stroke-width:1px;stroke:currentColor;opacity:0.4;stroke-dashoffset:0;transition:stroke 0.2s ease-in-out;", ":hover &{opacity:1;}", ":active &{fill:rgb(255,255,255,0.1);}"], SVGContainer, SVGContainer);

exports.IndicatorCircle = IndicatorCircle;

var ProgressCircle = _styledComponents["default"].circle.withConfig({
  displayName: "components__ProgressCircle",
  componentId: "sc-19b1rha-3"
})(function (_ref2) {
  var dismissIn = _ref2.dismissIn;
  return (0, _styledComponents.css)(["fill:transparent;stroke-dasharray:82px;stroke-width:1px;animation-delay:0.5s;animation-name:", ";animation-timing-function:linear;animation-fill-mode:forwards;animation-duration:", "ms;stroke:currentColor;stroke-dashoffset:", ";stroke-linecap:round;"], progress, dismissIn, dismissIn === 0 ? '-164px' : '-82px');
});

exports.ProgressCircle = ProgressCircle;
var Action = (0, _styledComponents["default"])(_Typography.Typography).withConfig({
  displayName: "components__Action",
  componentId: "sc-19b1rha-4"
})(["", ""], function (_ref3) {
  var onClick = _ref3.onClick;
  return onClick && (0, _styledComponents.css)(["cursor:pointer;&:hover{text-decoration:underline;}"]);
});
exports.Action = Action;
