// @aircall/tractor/lib/components/Toast/utils.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.makeToastVariant = exports.isDismissable = void 0;

/**
 * Check this figma link for more details regarding the theming
 * https://www.figma.com/file/KCJhOzhpVah4COL8rlZJxv/(WIP)-00.-Tractor---Phone?node-id=9%3A6
 */
var makeToastVariant = function makeToastVariant(_ref) {
  var variant = _ref.variant,
      theme = _ref.theme;
  var variants = {
    primary: {
      backgroundColor: theme.colors.primary.base,
      color: theme.colors.text.lighter
    },
    destructive: {
      backgroundColor: theme.colors.red.base,
      color: theme.colors.text.lighter
    },
    instructive: {
      backgroundColor: theme.colors.blue.base,
      color: theme.colors.text.lighter
    },
    warning: {
      backgroundColor: theme.colors.yellow.base,
      color: theme.colors.text.base
    }
  };
  return variants[variant];
};
/**
 * Check whether or not a toast can be automatically dismissed.
 */


exports.makeToastVariant = makeToastVariant;

var isDismissable = function isDismissable(dismissIn) {
  if (dismissIn === void 0) {
    dismissIn = 0;
  }

  return Number.isFinite(dismissIn) && dismissIn > 0;
};

exports.isDismissable = isDismissable;
